---
title: "Genomic Capacities for Reactive Oxygen Species Metabolism Across Marine Phytoplankton"
author:
- Naaman M. Omar:
    institute: mta
- Katherine Fleury:
    institute: [mta, ualberta]
- Brian Beardsall:
    institute: [mta, dalhousie]
- Ondřej Prášil:
    institute: algatech
- Douglas A. Campbell:
    institute: mta
    email: dcampbel@mta.ca
    correspondence: TRUE
institute:
  - mta: 'Department of Biology, Mount Allison University, 53 York St., Sackville NB, Canada, E4L 1C9.'
  - ualberta: 'Faculty of Veterinary Medicine, University of Calgary, 3280 Hospital Dr NW, Calgary, AB, T2N 4Z6.'
  - dalhousie: 'Faculty of Computer Science, Dalhousie University, 6050 University Ave, Halifax, NS, B3H 1W5.'
  - algatech: 'Center Algatech, Laboratory of Photosynthesis, Novohradska 237, Trebon, CZ 37981, Czech Republic.'
date: "`r format(Sys.Date())`"
output:
  bookdown::html_document2:
    code_folding: show
    keep_md: yes
    fig_caption: yes
    toc: TRUE
    toc_float: TRUE
    toc_depth: 6
    pandoc_args:
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
  bookdown::word_document2:
    code_folding: show
    keep_md: yes
    fig_caption: yes
    reference_docx: KnitTemplate.docx
    pandoc_args:
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
bibliography: "`r file.path('..', '..', 'ROS_Phytoplankton.bib')`"
csl: plos-one.csl
# header-includes: 
always_allow_html: true
editor_options: 
  markdown: 
    wrap: 72
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, message =  FALSE, warning = FALSE, fig.path = "../Figures/")
```

```{r packages and paths, include = FALSE}
library(tidyverse)
library(cowplot)
library(broom)
# library(chngpt)
library(ggpubr)
library(glue)
library(smatr)
#https://cran.r-project.org/web/packages/kableExtra/vignettes/awesome_table_in_html.html#installation
library(bookdown)
library(kableExtra)
options(kableExtra.auto_format = FALSE)
library(rcompanion)
library(corrplot)
library(knitr)
# library(PanVizGenerator)
library(ggiraphExtra)
library(AER)
googlesheets4::gs4_deauth() 
```

```{r assign filepaths, include = FALSE}
Figures <- file.path("..", "Figures")
DataIn <- file.path("..", "DataIn")
FigureMaterial <- file.path("..", "..", "FigureMaterial")
DataProcessed <- file.path("..", "DataProcessed")
DataProcessedModels <- file.path("..", "DataProcessed")
```

```{r read in data, include = FALSE}
MergedData <- readRDS(file.path(DataProcessed, "MergedData.Rds"))
TaxaColors <- readRDS(file.path(DataProcessed, "TaxaColours.Rds"))
WideData <- readRDS(file.path(DataProcessed, "WideData.Rds"))
OmesPairs <- readRDS(file.path(DataProcessed, "OmesPairs.Rds"))
```

```{r generate ROSMin and ROSMax, include = FALSE}
ROSMin <- MergedData %>% 
  select(HyPe_count, GeneModels_count) %>% 
  filter(HyPe_count != 0, GeneModels_count != 0) %>% 
  mutate(ratio = 1/GeneModels_count) %>% #1 is the smallest possible number of genes that could be identified
  select(ratio) %>%
  min() %>% 
  log10()

ROSMax <- MergedData %>% 
  select(HyPe_count, GeneModels_count) %>% 
  filter(HyPe_count != 0, GeneModels_count != 0) %>% 
  mutate(ratio = HyPe_count/GeneModels_count) %>% 
  select(ratio) %>%
  max() %>% 
  log10()

ROSRatioMax <- MergedData %>% 
  select(HyPe_count, GeneModels_count) %>% 
  filter(HyPe_count != 0, GeneModels_count != 0) %>% 
  mutate(ratio = HyPe_count/GeneModels_count) %>% 
  select(ratio) %>%
  max() 
```

```{r ln_log10_conversion, include = FALSE}
ln_log10_conversion <- 2.30258499 # Divide to convert ln to log10; Multiply to convert ln to log10
```

```{r generate ROSBoxPlotMax, include = FALSE}
ROSBoxPlotMax <- MergedData %>% 
  select(HyPe_count, GeneModels_count, Taxa) %>% 
  filter(HyPe_count != 0, 
         GeneModels_count != 0, 
         Taxa == "Diatom") %>% 
  mutate(ratio = (HyPe_count+1)/GeneModels_count) %>% 
  select(ratio) %>%
  max()

MyShapeComparisons <- list(c("Pennate", "Centric"))
MyColonyComparisons <- list(c("TRUE", "FALSE"))
MyFlagellaComparisons <- list(c("yes", "no"))

```

```{r define functions, include = FALSE}
`%notin%` <- Negate(`%in%`) #https://www.r-bloggers.com/2018/07/the-notin-operator/

# ScientificTen is a function that converts a numeric value to scienific format. x is the numeric value whereas sf is the number of significant figures. Default is 2. 
# ScientificTen <- function(x, sf = 2){
#   y <- format(signif(x, sf), scientific = TRUE) %>% as.character()
#   z <- str_split(y, pattern = "e")[[1]] 
#   if(z[2]<0){
#     return(glue(z[1],"×10","^",str_replace_all(z[2], "[[+]]", ""),"^"))
#   }
#   else{
#     return(glue(z[1],"×10","^",z[2],"^"))
#   }
# }

ScientificTen <- function(x, sf = 2){
  y <- format(signif(x, sf), scientific = TRUE) %>% as.character()
  z <- str_split(y, pattern = "e")[[1]] %>% as.numeric()
  a <- if_else(z[2] > -10 & z[2] < 10, last(z[2]), z[2])
  if(z[2]<0){
    return(glue(z[1],"×10","^",str_replace_all(a, "[[+]]", ""),"^"))
  }
  else{
    return(glue(z[1],"×10","^",a,"^"))
  }
}
```

# Abstract {.unnumbered}

Marine phytoplankton both produce and scavenge Reactive Oxygen Species (ROS), to support cellular processes, while limiting damaging reactions. Some prokaryotic picophytoplankton have, however, lost all genes encoding capacity to scavenge hydrogen peroxide,  and so rely upon co-occurring cells to scavenge hydrogen peroxide to tolerable levels. Such losses of metabolic function can only apply to those ROS, including hydrogen peroxide, which potentially traverse the cell membrane, before provoking excessive damaging intracellular reactions.
We therefore hypothesized that cell radius influences which elements of ROS metabolism can potentially be out-sourced to co-occurring cells. We investigated genomes and transcriptomes from diverse marine eukaryotic phytoplankton, ranging from 0.4 to 44 µm radius, to analyze the genomic allocations encoding enzymes metabolizing three distinct ROS.
Superoxide has high reactivity, short intracellular and extracellular lifetimes and limited membrane permeability. Genes encoding superoxide scavenging were ubiquitous across marine eukaryotic phytoplankton, but the fractional gene allocation decreased with increasing cell radius, consistent with superoxide metabolism served by a nearly fixed set of core genes, scavenging intracellular and extracellular superoxide pools, separated by the cell membrane.
Hydrogen peroxide has lower reactivity, longer intracellular and extracellular lifetimes and readily crosses cell membranes. Marine eukaryotic phytoplankton genomic allocations to hydrogen peroxide production decrease with increasing cell radius, more than do genomic allocations to scavenging of hydrogen peroxide, consistent with maintenance of ROS homeostasis in the face of slower diffusional losses of hydrogen peroxide from larger cells. Gene allocations to hydrogen peroxide metabolism are positively influenced by capacity for colony formation, suggesting more active hydrogen peroxide metabolism among the closely spaced cells of colonies compared to single cells; and by the presence of flagella,  suggesting an interaction between flagella and maintenance of ROS homeostasis.
Nitric Oxide has low reactivity, long intracellular and extracellular lifetimes and readily crosses cell membranes. Neither nitric oxide production nor scavenging genomic capacities changed with increasing cell radius.  Centric vs. pennate diatoms, however, show contrasting gene allocations to nitric oxide metabolism, consistent with diverse regulatory roles of nitric oxide across taxa.

# Introduction {.unnumbered}

Phytoplankton cells span a large size range, from picoplankton (<2µm), nanoplankton (2 to 20µm), microplankton (20 to 200µm) to macroplankton (200 to <2000µm) [@finkelPhytoplanktonChangingWorld2010]. Cell size interacts with multiple selective pressures, including cellular metabolic rate, light absorption, nutrient uptake, cell nutrient quotas, trophic interactions and diffusional exchanges with the environment  [@finkelPhytoplanktonChangingWorld2010; @andersenModellingEmergentTrophic2015; @finkelLightAbsorptionSize2001; @geiderSizeDependenceGrowth1986; @stromMicrozooplanktonGrazingCoastal2007]. Beyond simple size, cells of different shapes differ in surface area to volume ratio. For example, more elongated cells, such as pennate diatoms, have a larger surface area to volume ratio compared to more rounded cells, such as centric diatoms, of equivalent biovolume [@litchmanLinkingTraitsSpecies2010], which can in turn influence diffusional exchanges between cells and their environment. 

## Characteristics of Reactive Oxygen Species {.unnumbered}

Phytoplankton both produce and scavenge Reactive Oxygen Species (ROS), both within and outside the cell membrane. Some ROS readily cross the cell membrane, connecting intra- and extra-cellular pools. Other ROS rarely cross cell membranes and therefore intra- and extra-cellular pools are at least partially segregated (Table \@ref(tab:ROSDiffusion)).

```{r ROSDiffusion, echo = FALSE}
ROSDiffusion <- googlesheets4::read_sheet("https://docs.google.com/spreadsheets/d/1ds8brzxLzTitaVCv3Evf2extGqJfrKjQynyAVx5m7hg/edit#gid=663628435")

ROSDiffusion %>% 
  kable(caption = "Diffusion and Stability of Different ROS", digits = 32) %>%
  kable_classic()
```

Superoxide (O~2~^•−^), a radical anion generated through the monovalent reduction of O~2~ to O~2~^•−^ [@hayyanSuperoxideIonGeneration2016], is present in the oceans at concentrations of 10^-12^ to 10^-9^ [@diazProductionExtracellularReactive2018; @adesinaPhotochemicallyGeneratedNitric2021]. O~2~^•−^ is highly reactive [@fridovichOxygenToxicityRadical1998] with organic compounds including thiols [@winterbournThiolChemistrySpecificity2008] and with metals [@roeUptakeMechanismsInorganic2014; @roseInfluenceExtracellularSuperoxide2012]. As the first ROS in a sequential series of reductions of O~2~, O~2~^•−^ is a 'gateway' to production of other ROS. O~2~^•−^ is produced both inside and outside a cell, but shows limited diffusion across the hydrophobic cell membrane [@korshunovPotentialRolePeriplasmic2002]. Electron transfers to O~2~ from the photosynthetic and mitochondrial electron transport chains generate intracellular O~2~^•−^ as a by-product [@kozulevaMinimizingElectronFlow2020; @blokhinaReactiveOxygenSpecies2010; @asadaUnivalentReductionMolecular1974; @kozulevaEvaluationParticipationFerredoxin2010;@pospisilProductionReactiveOxygen2009; @pospisilRoleMetalsProduction2014]. Multiple oxidases (Table \@ref(tab:MetaData)) reduce O~2~ and generate either H~2~O [@solomonO2ReductionH2O2008], or alternately O~2~^•−^ and/or Hydrogen Peroxide (H~2~O~2~) [@messnerMechanismSuperoxideHydrogen2002; @mittlerOxidativeStressAntioxidants2002]. Biogenic production of extracellular ROS mediated by taxa ranging from heterotrophic bacteria to diatoms is significant in marine environments [@zhangExtensiveDarkBiological2016; @diazWidespreadProductionExtracellular2013; @diazProductionExtracellularReactive2018; @diazProductionExtracellularSuperoxide2018; @roseMeasurementImplicationsNonphotochemically2008; @schneiderSpeciesLevelVariabilityExtracellular2016; @sutherlandExtracellularSuperoxideProduction2019; @kustkaExtracellularProductionSuperoxide2005; @hanselDynamicsExtracellularSuperoxide2016]. Increasing cell density leads to decreased O~2~^•−^ production per cell [@sutherlandExtracellularSuperoxideProduction2019; @hanselTightRegulationExtracellular2019; @marshallSuperoxideProductionMarine2005]. Extracellular O~2~^•−^ production likely contributes to cell growth, in that the removal of extracellular ROS from cultures of the heterotrophic bacteria *Roseobacter* inhibits its growth [@hanselTightRegulationExtracellular2019]. O~2~^•−^ in coastal waters is primarily attributable to extracellular production by eukaryotic phytoplankton [@sutherlandSpatialHeterogeneityParticleAssociated2020], while *Prochlorococcus* MED4 shows the least extracellular production of O~2~^•−^ among microbes analyzed to date [@sutherlandExtracellularSuperoxideProduction2019]. O~2~^•−^ production patterns can vary even within a genus. [@schneiderSpeciesLevelVariabilityExtracellular2016].

Two known enzymes mediate conversion of O~2~^•−^ to H~2~O~2~; the ubiquitous dismutation of O~2~^•−^ catalyzed by diverse Superoxide Dismutases (SOD) or the less prevalent reduction of O~2~^•−^, catalyzed by Superoxide Reductase at the expense of metabolic reductant. O~2~^•−^ also dismutates spontaneously to produce H~2~O~2~ and O~2~ [@zafiriouChemistrySuperoxideIonradical1990], although @sutherlandDarkBiologicalSuperoxide2020 found that ~52% of dark O~2~^•−^ production instead likely undergoes oxidation back to O~2~.  Extracellular production of O~2~^•−^ thus indirectly contributes to extracellular H~2~O~2~ [@wuttigPathwaysSuperoxideDecay2013; @sutherlandExtracellularSuperoxideProduction2019; @scanlanEcologicalGenomicsMarine2009; @schneiderSpeciesLevelVariabilityExtracellular2016].

H~2~O~2~ is an uncharged compound, present in the ocean at concentrations of 10^-9^ to 10^-6^ mol L^-1^ [@diazProductionExtracellularReactive2018], depending upon location and conditions including temperature, light level, depth of the mixed layer and the concentration of total dissolved organic carbon [@yuanDistributionHydrogenPeroxide2001]. In aquatic systems, H~2~O~2~ has lifetimes of hours to days (Table \@ref(tab:ROSDiffusion)) [@lesserOXIDATIVESTRESSMARINE2006; @diazProductionExtracellularReactive2018]. H~2~O~2~ passively traverses cell membranes [@halliwellFreeRadicalsBiology1999], primarily through aquaporins [@bienertAquaporinfacilitatedTransmembraneDiffusion2014; @almasalmehStructuralDeterminantsHydrogen2014; @wangCharacterizationAquaporindrivenHydrogen2020; @millerAquaporin3MediatesHydrogen2010], allowing exchange of intracellular and extracellular pools of H~2~O~2~, although cells can maintain a concentration gradient between internal and external H~2~O~2~ [@seaverHydrogenPeroxideFluxes2001].

H~2~O~2~ is acutely toxic to most cells in the range of 10^-5^ to 10^-4^ mol L^-1^ [@halliwellFreeRadicalsBiology1999] (Table \@ref(tab:ROSDiffusion)), and contributes significantly to the redox cycling of copper and iron [@milleroRateReductionCopper1991; @moffettReactionKineticsHydrogen1987]. H~2~O~2~ can react with thiols and methionine [@winterbournThiolChemistrySpecificity2008] and can interfere with gene expression [@fedurayevHydrogenPeroxideParticipates2018]. Cytotoxic effects of H~2~O~2~, including lipid damage, are primarily caused by H~2~O~2~ dismutating into the hydroxyl radical, which is strongly oxidative [@lesserOXIDATIVESTRESSMARINE2006]. H~2~O~2~ concentrations in seawater follow a diurnal cycle with a peak at mid-day [@morrisDielRegulationHydrogen2016; @averyHydrogenPeroxideBermuda2005; @yuanDistributionHydrogenPeroxide2001], suggesting significant direct or indirect photochemical or photobiological generation of H~2~O~2~. Although some marine H~2~O~2~ production is indeed linked directly to biological processes, they are not the dominant source of H~2~O~2~ in seawater [@averyHydrogenPeroxideBermuda2005]. H~2~O~2~ decomposes spontaneously, though slowly, into water and oxygen [@senguptaControllingNonCatalyticDecomposition2004].

Significant biotic H~2~O~2~ detoxification in aqueous systems may be attributed to heterotroph mediated activity, although heterotrophs do not contribute much of the H~2~O~2~ production [@bondHeterotrophicBacteriaExhibit2020].   
*Prochlorococcus* and some strains of *Synechococcus* lack genes encoding catalase and are therefore difficult to grow in axenic cultures, particularly at high cell densities. Growth and survival of these strains improves when co-cultured with other 'helper' bacteria which carry genes for catalase [@zinserCrossprotectionHydrogenPeroxide2018; @morrisDependenceCyanobacteriumProchlorococcus2011; @morrisFacilitationRobustGrowth2008; @coeSurvivalProchlorococcusExtended2016]. These culture results were found at *Prochlorococcus* cells densities, somewhat higher than typical ocean levels of 1 × 10^5^ to 3 × 10^5^ *Prochlorococcus* cells ml^−1^ [@partenskyProchlorococcusMarinePhotosynthetic1999], where auto-intoxication from metabolism of the *Prochlorococcus* cell population might be less prevalent [@omarDiffusionalInteractionsMarine2022]. In the open ocean, H~2~O~2~ levels generally remain below the cytotoxic threshold of ~1 x 10^-5^ M, perhaps partly through activity of co-occuring helper microorganisms that retain capacity to produce catalase [@zinserCrossprotectionHydrogenPeroxide2018; @morrisDielRegulationHydrogen2016].

Despite its radical nature and ability to react with biomolecules, ^•^NO functions widely as a signaling molecule  [@vardiStressSurveillanceSystem2006; @thomsonEcophysiologyBrineDinoflagellate2000]. Optimal *in vivo* concentrations of ^•^NO for phytoplankton growth vary significantly across taxa, from 10^−8^ to 10^−9^ mol extracellular ^•^NO L^-1^ [@zhangStudyPatternsChemical2005]. Overexpression of ^•^NO producing genes in diatoms results in reduced growth, photosynthetic activity and ability to adhere to surfaces, and thus likely a decrease in biofilm formation [@vardiDiatomGeneRegulating2008]. @vardiDiatomGeneRegulating2008 further suggest that ^•^NO controls cell motility in that diatoms increase ^•^NO production under stressful conditions thereby decreasing cellular adhesion, freeing the cell to find a more suitable habitat to which to adhere. ^•^NO may also react with reduced glutathione (GSH) to form S-nitrosoglutathione (GSNO) [@delrioReactiveOxygenSpecies2009; @jahnovaSNitrosoglutathioneReductaseMaster2019].
Most cellular damage mediated by ^•^NO is attributed to the reaction of ^•^NO with O~2~^−^ to produce Peroxinitrite (ONOO^−^). The reaction of ^•^NO with O~2~^•−^ to produce ONOO^-^ is limited by the concentration of ^•^NO and is not a major ^•^NO sink in seawater [@adesinaPhotochemicallyGeneratedNitric2021]. 

Other important ROS, Singlet Oxygen (^1^O~2~), Peroxynitrite (ONOO^-^) and Hydroxyl Radical (HO^•^) are not known to be directly produced nor scavenged by enzymatic processes [@collinChemicalBasisReactive2019; [@marusawaHydroxylRadicalStrong2002; @gutteridgeReactivityHydroxylHydroxyllike1984; @mcgillChapterOxidantStress2013; @daviesProteinsDamagedOxygen1987; @halliwellFreeRadicalsBiology1999]. In seawater, HO^•^ is present in concentrations of 10^−18^ to 10^−15^ mol L^-1^, and has a diffusion distance of only ~4.5 nm before destruction with a lifetime of µs [@diazProductionExtracellularReactive2018; @lesserOXIDATIVESTRESSMARINE2006]. Because to the high reactivity of HO^•^, it is unlikely that there are any scavengers dedicated to HO^•^ specifically [@halliwellFreeRadicalsBiology1999], although reactions with Dissolved Organic Matter non-specifically scavenge extracellular HO^•^ [@brezonikNitrateInducedPhotolysisNatural1998].

## The Black Queen Hypothesis {.unnumbered}
The Black Queen Hypothesis states that loss of function mutations may proceed so long as some interacting community members retain the function, and the function can occur outside a given cell [@morrisBlackQueenHypothesis2012]. The Black Queen Hypothesis was formulated on the basis of *Prochlorococcus* and H~2~O~2~. *Prochlorococcus* (the beneficiary) lost the gene encoding the enzyme *katG* which scavenges H~2~O~2~ . Instead *Prochlorococcus* allows H~2~O~2~ outwards across the cell membrane to be dealt with by community members retaining the capacity to scavenge H~2~O~2~, thus saving *Prochlorococcus* the cost of maintaining the genes and metabolism for scavenging H~2~O~2~ [@morrisDependenceCyanobacteriumProchlorococcus2011; @morrisBlackQueenHypothesis2012].

## Hypotheses and Significance {.unnumbered}

Given that ROS show differential abilities to cross cell membranes, and have widely different diffusion distances before destruction, we sought to study whether cell radius, or diatom cell shape influence genomic allocations to ROS production and scavenging across diverse marine phytoplankters.

```{hypothesis SupOx}
**Cell radius across phytoplankton taxa does not influence the fraction of their total gene content encoding O~2~^•−^ production,  nor scavenging. **
O~2~^•−^ is not readily able to cross biological membranes [@korshunovPotentialRolePeriplasmic2002], so there are separate pools of intracellular and extracellular O~2~^•−^, diffusional losses of O~2~^•−^ from cells are limited and will be little influenced by cell size.
```

```{hypothesis HyPeNitOxRad}
**Large phytoplankton allocate a smaller fraction of their total gene content to H~2~O~2~ and ^•^NO production and a larger fraction of their total gene content to H~2~O~2~ and ^•^NO scavenging.**
  
H~2~O~2~ and ^•^NO have relatively low reactivity, with long intracellular and extracellular lifetimes leading to long potential diffusion paths before destruction.  Both H~2~O~2~ and ^•^NO are uncharged and readily cross cell membranes (Table \@ref(tab:ROSDiffusion)).
Large cells have longer intracellular diffusional paths and a lower surface to volume ratios than do smaller cells [@finkelPhytoplanktonChangingWorld2010]. Large cells are thus less prone to diffusional losses of intracellular H~2~O~2~ and ^•^NO.
We therefore hypothesize that large phytoplankton taxa will allocate a smaller fraction of their gene contents to H~2~O~2~ and to ^•^NO production, to maintain H~2~O~2~ and ^•^NO homeostasis in the face of slower diffusional losses of H~2~O~2~ or ^•^NO out of the cells to the environment.  In contrast, loss of function mutations on enzymes that scavenge H~2~O~2~ and ^•^NO would be more deleterious in large cells than in smaller cells [@morrisBlackQueenHypothesis2012; @morrisDependenceCyanobacteriumProchlorococcus2011].
```

```{hypothesis HyPeNitOxDiatom}
**Pennate Diatoms allocate a larger fraction of their total gene content to H~2~O~2~ and ^•^NO production, and a smaller fraction of their total gene content to H~2~O~2~ and ^•^NO scavenging than do Centric Diatoms.**

Pennate Diatoms maintain a small minimum radii even at large biovolumes due to their elongated shape [@mitchellRoleDiatomNanostructures2013]. As a result, the cell shape of pennate diatoms allows for more diffusion of H~2~O~2~ and ^•^NO across the cell membrane due to the short mean diffusion paths to the cell surface and high surface area to volume ratio. Noting that H~2~O~2~ and ^•^NO are permeable across biological membranes (Table \@ref(tab:ROSDiffusion)), we hypothesize that pennate diatoms will allocate a larger fraction of their total gene content to H~2~O~2~ and ^•^NO production compared to centric diatoms. In contrast we hypothesize that pennate diatoms will allocate a smaller fraction of their gene content to H~2~O~2~ and ^•^NO scavenging, compared to centric diatoms.
```

```{hypothesis Colony}
**Colony forming phytoplankton have a smaller fraction of their total gene content encoding H~2~O~2~ and ^•^NO production, and a larger fraction of their total gene content encoding  H~2~O~2~ and ^•^NO scavenging. **
Cell spacing in colony forming phytoplankton is so small that the diffusional spheres of H~2~O~2~ or ^•^NO diffusing outwards from cells overlap with nearby cells [@omarDiffusionalInteractionsMarine2022], thereby shifting the requirements to maintain homeostasis within cells of a colony. 
```

```{hypothesis Flagella}
**Flagellated phytoplankton have a larger fraction of their total gene content encoding  H~2~O~2~ and ^•^NO production, and a smaller fraction of their total gene content encoding  H~2~O~2~ and ^•^NO scavenging.** 
Increased motility in flagellated cells, allows movement away from cytotoxic levels of  H~2~O~2~ and ^•^NO, substituting for scavenging. 
```

# Methods {.unnumbered}

XXX Reviewer 1 would like a more detailed MatMet

## Data Dictionary {.unnumbered}

Table \@ref(tab:DataDictionary) contains a data dictionary of variable names used in our analysis, their definitions and locations in code and data objects. 

## Bioinformatic Pipeline {.unnumbered}

We downloaded Genomes and/or Transcriptomes of `r MergedData$FileName %>% unique() %>% length()` diverse marine phytoplankton (Table \@ref(tab:MetaData)) from the National Center for Biotechnology Information (NCBI) [@DatabaseResourcesNational2016]; Joint Genome Institute (JGI) [@grigorievGenomePortalDepartment2012; @nordbergGenomePortalDepartment2014]; iMicrobe [@youens-clarkIMicrobeToolsDatadriven2019], European Nucleotide Archive (ENA) [@leinonenEuropeanNucleotideArchive2011]; pico-PLAZA [@vandepoelePicoPLAZAGenomeDatabase2013], 1000 Plants (1KP) [@matasciDataAccess0002014]; or the Reef Genomics Database [@liewReefgenomicsOrgRepository2016] (Figure \@ref(fig:BioinformaticSummary)). 

We implemented an automated pipeline using Snakemake [@molderSustainableDataAnalysis2021] to pass gene sequences from downloaded genomes or transciptomes, in .fasta format, to the eggNOG 5.0 database [@huerta-cepasEggNOGHierarchicalFunctionally2019], and then used eggNOG-Mapper 2.0.6 [@huerta-cepasFastGenomeWideFunctional2017; @cantalapiedraEggNOGmapperV2Functional2021] and the DIAMOND algorithm [@buchfinkFastSensitiveProtein2015], to annotate potential orthologs in each analyzed genome or transcriptome, using the following parameters: seed_ortholog_evalue = 0.001, seed_ortholog_score = 60, tax_scope = "auto", go_evidence = "non-electronic", query_cover = 20 and subject_cover = 0.
The annotation generated for each gene model included (when available): the name of the matching ortholog (coded by eggNOG as 'seed_eggNOG_ortholog'); E-value (coded by eggNOG as 'seed_ortholog_evalue'); Score (coded by eggNOG as 'seed_ortholog_score'); EC number (coded by eggNOG as 'EC'); Kegg Orthology (KO) number (coded by eggNOG as 'KEGG_ko'); Kegg Pathway (coded by eggNOG as 'KEGG_Pathway'); Kegg Module (coded by eggNOG as 'KEGG_Module'); Kegg Reaction (coded by eggNOG as 'KEGG_Reaction'); Kegg Reaction Class (coded by eggNOG as 'KEGG_rclass'); the predicted protein family (coded by eggNOG as 'PFAMs'); Gene Ontology (GO) annotation (coded by eggNOG as 'Gos'); as well as a description from eggNOG of the source organism of the matching ortholog (coded by eggNOG as 'best_og_desc').
Note that the output from sequences passed through the eggNOG 5.0 database are non-supervised orthology annotations, and may be subject to error if the underlying eggNOG annotation was inaccurate, or for functionally divergent orthologous gene sequences.
The output of automatically annotated orthologs, from each genome or transcriptome, from the bioinformatic pipeline was compiled into one file `CombinedHits.csv` (to be submitted to the DRYAD database) (Figure \@ref(fig:BioinformaticSummary)).

```{r BioinformaticSummary, fig.cap = BioinformaticSummary_cap, fig.align = 'centre',out.width='200%', out.height='200%', echo = FALSE}
BioinformaticSummary_cap <- glue("**Summary Flowchart of Methods.**", sep = " ")

# Original figure can be found in ../FigureMaterial/NaamanMethods.pptx
BioinformaticSummary <- knitr::include_graphics(file.path(FigureMaterial,"NaamanMethods.jpg"))
BioinformaticSummary


```

## Overview of Data Analysis {.unnumbered}

`CombinedHits.csv` was imported into a data frame (coded as 'CombinedHits') for analysis using R [@rcoreteamLanguageEnvironmentStatistical2019] running under RStudio [@rstudioteamRStudioIntegratedDevelopment2015], using the 'tidyverse' [@wickhamTidyverseEasilyInstall2017], 'broom' [@robinsonBroomConvertStatistical2019], 'magrittr' [@bacheMagrittrForwardPipeOperator2014], 'dplyr' [@wickhamDplyrGrammarData2018], 'chngpt' [@fongChngptThresholdRegression2017], 'rcompanion' [@mangiaficoRcompanionFunctionsSupport2020], 'gmodels' [@warnesGmodelsVariousProgramming2018], 'stats' [@rcoreteamLanguageEnvironmentStatistical2019] and 'smatr' [@wartonSmatrPackageEstimation2012] packages.
Graphics and tables were generated using the 'ggplot2' [@wickhamGgplot2ElegantGraphics2016], 'cowplot' [@wilkeCowplotStreamlinedPlot2019], 'glue' [@hesterGlueInterpretedString2018], 'kableExtra' [@zhuKableExtraConstructComplex2019], 'corrplot' [@weiPackageCorrplotVisualization2017], 'ggfortify' [@tangGgfortifyUnifiedInterface2016; @horikoshiGgfortifyDataVisualization2018], 'AER' [@kleiberAERAppliedEconometrics2020] and 'ggforce' [@pedersenGgforceAcceleratingGgplot22021] packages (Figure \@ref(fig:BioinformaticSummary)).
Citations were managed using the Zotero (www.zotero.org) open access reference manager connected to RStudio using the 'citr' [@austCitrRStudioAddin2018] package.
The Zotero library of citations is available at (https://www.zotero.org/groups/2333131/ros_phytoplankton).
Formatted outputs were generated from RMarkdown files using the 'knitr' [@xieKnitrComprehensiveTool2014a; @xieDynamicDocumentsKnitr2015; @xieKnitrGeneralPurposePackage2018] and 'bookdown' [@xieBookdownAuthoringBooks2019] packages. 

In parallel we assembled metadata from the literature and culture collection databases for each genome or transcriptome source organism, including the cell radii in µm from 100% of organisms; colony formation for 84% of organisms; cell shape from diatoms from 100% of diatoms; presence or absence of flagella as an index of potential motility coded as 'Flagella' from 100% of organisms, the genome size from all genomes and the total number of predicted gene models from 80% of organisms (Table \@ref(tab:MetaData)); all stored in `CellGenomeMetrics.csv` (Figure \@ref(fig:BioinformaticSummary)).
For organisms for which only transcriptomes were available, we only included datasets for which the total number of detected different transcripts was available, as a proxy for the total number of predicted genes.
Strains of brackish origin were included  but we did not include true freshwater strains in our analyses.

We compared the Enzyme Commission Number (EC number) to the BRENDA enzyme database [@changBRENDAELIXIRCore2021] to identify enzymes metabolizing H~2~O~2~, O~2~^•−^ or ^•^NO *in vivo* (annotated by BRENDA as 'natural product' or 'natural substrate') (Table \@ref(tab:ROSEnzymeList); Figure \@ref(fig:BioinformaticSummary)).
We then used the EC Number to filter 'CombinedHits' to generate a subset containing only those orthologs encoding enzymes mediating metabolism of H~2~O~2~ (Production or Scavenging), O~2~^•−^ (Production or Scavenging) and ^•^NO (Production or Scavenging).
Genes that code for the production of non-enzymatic ROS scavengers such as carotenoids and tocopherols were not included in subsequent analyses. XXX

From the subset of 'CombinedHits' containing information on ROS metabolizing enzymes, we grouped orthologs together by EC number and their Kegg Orthology number (KO number) and determined the number of occurrences of individual orthologs encoding each EC number (or KO number when EC number was not available) in a given organism.
We merged this data subset with `CellGenomeMetrics.csv` to generate a dataset of the total number of genes encoding a ROS; EC numbers; KO Numbers; and Gene Ontology annotations; that code for the production of a ROS metabolizing enzyme, as defined by the EC or KO number, along with characters of the source organism (combined into 'MergedData').

H~2~O~2~, O~2~^•−^ and ^•^NO are vastly different in reactivity, stability, diffusion distance, effects on biomolecules and roles in cell signaling (Table \@ref(tab:ROSDiffusion)), so it would not be a valid to pool counts of orthologs encoding production or scavenging of all these diverse ROS.
We therefore generated the total gene counts coding for the production or scavenging of a given ROS in a given organism, which were used to generate Poisson or Quasi-Poisson regressions (Figure \@ref(fig:BioinformaticSummary)).

## Data Validation & Justification of Statistical Analyses {.unnumbered}

Data from both genomes and transcriptomes were used in this analysis to gain wider representation from more taxa (Figure \@ref(fig:RadTaxaviolin)). Data from the taxa with the largest radii was derived wholly from transcriptomes. Aside from the prokaryote genomes sourced solely from within the 45° north south latitude band, the sampled phytoplankton did not exhibit taxonomic biases in source latitude of isolation, but were primarily coastal (Figure \@ref(fig:LongLatMap)). Otherwise where possible the total number of genes encoding ROS and the total number of genes in the organism derived from genomic data were used for subsequent analyses. For about 40 organisms we had both genomic and transcriptomic data, which we used to test assumptions on data distributions (Figure \@ref(fig:OmesPlot)). When both genomic and transcriptomic data were available, we only use genomic data in subsequent analyses (Table \@ref(tab:MetaData)). We use transcriptomes to fill out gaps in data when genomes were not available. We validated the gene annotations generated by the snakemake bioinformatic pipeline, by comparing the total number of genes encoding ROS metabolism data from a subset of 'CombinedHits.csv' to the total number of genes encoding ROS metabolism data from a manually annotated dataset generated during a pilot project [@fleuryReactiveOxygenDetoxification2019; @omarReactiveOxygenProduction2020]. 

```{r ManualAutoCompareChi, include = FALSE}
ManualAutoCompareData <- read_csv(file.path(DataIn, "SnakeManualAutomatedCountCompare.csv")) %>%
  drop_na()

# ManualAutoCompareChi <- chisq.test(ManualAutoCompareData$AutomatedECKO_count, ManualAutoCompareData$Manual_count, correct = TRUE) %>%
#   tidy()

ValidationCorTest <- cor.test(ManualAutoCompareData$AutomatedECKO_count, ManualAutoCompareData$Manual_count) %>% 
  tidy()


# Should I switch to a cor.test instead of a manual count?

# ManualAutoCompareChi %>%
#   kable(caption = "Chi-square test comparing manually and automatically annotated gene count data.") %>%
#   kable_classic()
```


Figure \@ref(fig:ManualAutoCompare) shows a significant correlation (Correlation of `r signif(ValidationCorTest$estimate, 2)`, p = `r ScientificTen(ValidationCorTest$p.value, 2)`) between manually generated 'ROSGene_count' and the automated 'ROSGene_count' from the snakemake pipeline.

From the 'CombinedHits' data frame, we filtered out some enzymes whose BRENDA annotation of 'natural product' and 'natural substrate' was questionable, in particular:

- Superoxide oxidase (EC:1.10.3.17) carries a BRENDA annotation of 'natural product' for O~2~^•−^, despite the BRENDA citation stating that O~2~^•−^ production from superoxide oxidase was only documented *in vitro* with an excess of ubiquinone [@lundgrenScavengingSuperoxideMembranebound2018].

- D-amino-acid oxidase was removed from the H~2~O~2~ production calculations as the enzyme does not produce H~2~O~2~ *in vivo* [@matlashovHowMuchH2O22014].

Some enzymes were listed for a given substrate under both *natural product* and *natural substrate*, as for example when a reaction is reversible.
Bacterial non heme ferritin is listed under H~2~O~2~ production and scavenging as it produces H~2~O~2~ in the first of a two-step reaction and scavenges H~2~O~2~ in the second step [@bou-abdallahFunctionalityThreeSiteFerroxidase2014].

```{r GeneModelsRadPlot, fig.cap = GeneModelsRadPlot_cap, echo=FALSE}
GeneModelsRadPlot <- MergedData %>%
  # filter(Taxa != "Prokaryote") %>%
  filter(GeneModels_count != "0") %>%
  ggplot(aes(x=log_Radius_um, y= log_GeneModels_count)) +
  ggConvexHull::geom_convexhull(alpha = 0.2, aes(fill = Phylum, color = Phylum)) +
  geom_point(aes(shape = Ome, colour = Phylum), size = 3)+
  # ggalt::geom_encircle(aes(color = Phylum), expand = 0) +
  scale_colour_manual(values =TaxaColors)+
  scale_fill_manual(values =TaxaColors)+
  scale_shape_manual(values = c(19,1)) +
  theme_bw()+
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text=element_text(size=12),
        axis.title=element_text(size=14),
        aspect.ratio = 1,
        legend.text=element_text(size=10),
        legend.title = element_text(size=12)) +
  labs(x = ~log[10] ~ "(Median Cell Radius (µm))",
       y = ~log[10] ~ "(Total Number of Genes)")

GeneModelsRadPlot

GeneModelsRadPlot_cap <- glue("**Comparison of log~10~ of the total number of genes in an organism ('log_GeneModels_count') to log~10~ of the median cell radius in µm ('log_Radius_um').**", 
                              "Colour corresponds to the taxonomic lineage ('Phylum'), whereas symbol shape corresponds to the source of the data, whether Genome or Transcriptome ('Ome').", 
                              "Citations for data sources are in Supplementary Table S3.",
                              .sep = " ")

ggsave(paste("Figure2",".png",sep = ""), plot = GeneModelsRadPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r generate PoissonDistStats, include = FALSE}
PoissonDistStats <- MergedData %>% 
  select(FileName, SupOx, SupOx_count, HyPe, HyPe_count, NitOx, NitOx_count, Phylum) %>%
  unique() %>%
  pivot_longer(c(SupOx, HyPe, NitOx), names_to = "ROS", values_to = "ProdScav") %>% 
  pivot_longer(c(SupOx_count, HyPe_count, NitOx_count), names_to = "ROS_countType", values_to = "ROS_count") %>%
  select(FileName, ROS, ProdScav, ROS_countType, ROS_count, Phylum) %>%
  filter(!is.na(ProdScav)) %>%
  group_by(ROS, ProdScav) %>%
  # .$ROS_count %>%
  rstatix::shapiro_test(ROS_count) 
```

Figure \@ref(fig:PoissonDist) shows that the total number of genes encoding the metabolism of O~2~^•−^, H~2~O~2~ or ^•^NO are not normally distributed (Shapiro-Wilk Test [@shapiroAnalysisVarianceTest1965] with a p value of `r ScientificTen(PoissonDistStats$p[5])` for O~2~^•−^ scavenging, `r ScientificTen(PoissonDistStats$p[1])` for H~2~O~2~ production, `r ScientificTen(PoissonDistStats$p[2])` for H~2~O~2~ scavenging, `r ScientificTen(PoissonDistStats$p[3])` for ^•^NO production and  `r ScientificTen(PoissonDistStats$p[4])` for ^•^NO scavenging), and follows a Poisson distribution. Therefore, for subsequent analyses we used Poisson or Quasi-Poisson regressions to compare the total number of genes that encode the production or scavenging of O~2~^•−^, H~2~O~2~ or ^•^NO, with an offset of the total number of genes in the organism, to log~10~ of the median cell radius in µm.
Quasi-Poisson regressions were used when the Poisson regression was over-dispersed (dispersion > 1, p < 0.05) as determined by the 'AER' package [@kleiberAERAppliedEconometrics2020]. Because our data followed a Poisson distribution (Figure \@ref(fig:PoissonDist)), a Poisson regression followed by a chi-squared test or a Quasi-Poisson regression followed by an F test, is statistically when obtaining the p-value [@AnovaGlmFunction]. An alpha value of ≤0.05 was used as the threshold for statistical significance of regressions.

The total number of genes in each organism, increased with the median cell radius, and also varied among the taxonomic lineages (coded as Phylum) (Figure \@ref(fig:GeneModelsRadPlot)). Taxonomic lineage, in turn, interacts strongly with the median cell radius. For our analyses, we sought to detect effects of cell radius upon the fraction of total genes encoding ROS metabolism. We therefore normalized the total number of genes encoding the production or scavenging of H~2~O~2~, O~2~^•−^ or ^•^NO to 'GeneModel_count' to account for the general increase in 'GeneModel_count' with increasing the median cell radius. Because of the strong interaction between the median cell radius and taxonomic lineage (Figure \@ref(fig:RadTaxaviolin)), we did not include Phylum as a co-variate in our subsequent regressions of normalized gene counts vs. median cell radius. Thus, we did not analyze specific influences of Phylum. 

To investigate the influence of colony formation, the presence of flagella or diatom cell shape (pennate or centric), upon the total number of genes that encode the metabolism of H~2~O~2~, O~2~^•−^ or ^•^NO, with an offset of the total number of genes, we used a Wilcoxon test from the 'ggpubr' [@kassambaraGgpubrGgplot2Based2018] package to compare organisms from which we had data. 

# Results and Discussion {.unnumbered}

## Superoxide {.unnumbered}

```{r read in SupOx Models, include = FALSE}
SupOxRad <- readRDS(file.path(DataProcessedModels, "SupOxRad.Rds"))
# OchrophyteSupOxRad <- readRDS(file.path(DataProcessedModels, "OchrophyteSupOxRad.Rds"))
SupOxRadProk <- readRDS(file.path(DataProcessedModels, "SupOxRadProk.Rds"))
# SupOxLat <- readRDS(file.path(DataProcessedModels, "SupOxLat.Rds"))
# SupOxLatProk <- readRDS(file.path(DataProcessedModels, "SupOxLatProk.Rds"))
SupOxMerged <- readRDS(file.path(DataProcessed, "SupOxMerged.Rds"))
```

Neither prokaryotic nor eukaryotic phytoplankton had any sequences annotated that matched with enzymes that specifically code for the production of O~2~^•−^. All prokaryotic (Data not shown) and eukaryotic (Figure \@ref(fig:EukSupOxDotPlot)) phytoplankton (with the exception of a single transcriptome from *Micromonas polaris*) have genes encoding O~2~^•−^ scavenging enzymes, as they all have genes encoding the ubiquitous enzyme SOD.

XXX Reiterate that non enzymatic processes may affect the production of O~2~^•−^. These include environmental stresses (reviewer #1 and #2 comment).

Occurences of genes encoding the enzyme Superoxide Oxidase (SOO, EC:1.10.3.17) were rare, but present in a few diatom species (*Leptocylindrus danicus*, *Chaetoceros curvicetus* and *Thalassiosira minuscula CCMP1093.*) and prokaryotes (*Crocosphaera spp.*). Occurences of genes encoding the enzyme Superoxide Reductase (SOR, EC:1.15.1.2) were not detected in prokaryotes, and were detected in the diatoms, *Pseudo-nitzschia fradulenta WWA7* and *Seminavis robusta D6*, and the haptophyte *Pleurochrysis carterae CCMP456*.

Although records of enzymes producing O~2~^•−^ exist in the literature [@diazNADPHdependentExtracellularSuperoxide2019], in the marine phytoplankton genomes and transcriptomes that we analysed, we did not detect any genes that encode for enzymes (Table \@ref(tab:ROSEnzymes)) that specifically produce O~2~^•−^ based on the BRENDA allocation of O~2~^•−^ scavenging. It is however worth noting the presence of NADPH Oxidase (NOX) which can produce either H~2~O~2~ or O~2~^•−^ depending on the NOX isoform. NOX is included as a H~2~O~2~ producer in compliance with the BRENDA allocation of the enzyme (Table \@ref(tab:ROSEnzymes)). Further analyses may be able to search through the detected NOX isoforms to identify whether they are from the isoform that produces O~2~^•−^. Sequences that are also very similar to Glutathione Reductase (GR) have also been documented to produce enzymes that produce extracellular O~2~^•−^ in the diatom *Thalassiosira oceanica* [@diazNADPHdependentExtracellularSuperoxide2019]. GR sequences were annotated universally across phytoplankton genomes in previous analyses [@fleuryReactiveOxygenDetoxification2019; @omarReactiveOxygenProduction2020]. Given that @diazNADPHdependentExtracellularSuperoxide2019 found that these sequences were so similar to GR that they could not tell them apart, it is possible that O~2~^•−^ do exist within the genomes of these phytoplankton. In either case, we would not expect phytoplankton to differentially allocate a changing fraction of their total gene content to O~2~^•−^ production as O~2~^•−^ is not membrane diffusible. The idea that phytoplankton need to maintain working extracellular concentrations of O~2~^•−^ has been speculated however, noting that decreasing the extracellular concentration of O~2~^•−^ can hinder cell growth [@sutherlandExtracellularSuperoxideProduction2019]. @sutherlandExtracellularSuperoxideProduction2019 further explains that the downregulation of Superoxide Dismutase (SOD, EC:1.15.1.1) genes at peak light levels by *Prochlorococcus* [@mella-floresProchlorococcusSynechococcusHave2012] would allow *Prochlorococcus* to maintain 'working levels' of extracellular O~2~^•−^.

XXX fix statement on GR like sequence producing H~2~O~2~.

The absence of transcripts encoding SOD from the *Micromonas polaris* transcriptome is likely due to low expression of SOD at the time that the mRNA was harvested for sequence analyses. Given that the O~2~^•−^ is not membrane diffusible, intracellularly produced O~2~^•−^ would have to be scavenged or would react with other parts of the cell, as a result, organisms that do not have the ability to scavenge O~2~^•−^ would have reduced fitness. It would therefore be costly for marine phytoplankton to lose the genomic capacity to scavenge O~2~^•−^.

To our knowledge, our results though rare, are the first detection of the occurence of SOO and SOR in marine phytoplankton, and would need confirmation by Multiple Sequence Alignment. Multiple Sequence Alignment would answer whether our results are from errors in annotation from the non-supervised grouping of orthologs (genes) by eggNOG, or may indeed be sporadically present in the Genomes or Transcriptomes of some phytoplankton, perhaps as a result of Horizontal Gene Transfer.

Trends may yet appear in genomic allocations to O~2~^•−^ scavenging based on the metallo-types of SOD. For example, @fleuryReactiveOxygenDetoxification2019 found that pico-prasinophytes use Mn-SOD instead of Fe-SOD. One could analyze whether a similar trend is visible across a wider range of 'Taxa'.


### Influence of Cell Size on Gene Counts for Enzymes Metabolizing O~2~^•−^ {.unnumbered}

```{r generate SupOxRadPlot, include = FALSE}
SupOxRad$Includes <- c("Eukaryote Regressions")
SupOxRadProk$Includes <- c("Eukaryotes + Prokaryotes")
# DiatomSupOxRad$Includes <- ("Diatom")

SupOxRadPlots <- rbind(SupOxRad, SupOxRadProk) %>%
  unnest(c(offset_model_regression), names_sep = "_") %>%
  unnest(c(flagcol_offset_model_regression), names_sep = "_") %>%
  pivot_wider(., names_from = c(flagcol_offset_model_regression_term), values_from = c(flagcol_offset_model_regression_estimate, flagcol_offset_model_regression_std.error, flagcol_offset_model_regression_statistic, flagcol_offset_model_regression_p.value)) %>%
  pivot_wider(., names_from = c(offset_model_regression_term), values_from = c(offset_model_regression_estimate, offset_model_regression_std.error, offset_model_regression_statistic, offset_model_regression_p.value)) %>%
  unnest(cols = c(data)) %>%
  filter(Includes != "Eukaryotes + Prokaryotes") %>%
  unique() %>%
  ggplot() +
  geom_point(aes(x = log_Radius_um, y = log10(SupOx_count/GeneModels_count), color = Phylum)) + 
  geom_point(aes(x = log_Radius_um, y = log10(SupOx_count/GeneModels_count), color = Phylum), data = MergedData %>% filter(Phylum == "Prokaryote", SupOx == "Scavenging")) + 
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = `flagcol_offset_model_regression_estimate_(Intercept)`/ln_log10_conversion, 
                  slope = `flagcol_offset_model_regression_estimate_.$log_Radius_um`/ln_log10_conversion, 
                  linetype = flagcol_offset_regression_type), 
              color = "black") +
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = (`flagcol_offset_model_regression_estimate_(Intercept)` + `flagcol_offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`flagcol_offset_model_regression_estimate_.$log_Radius_um` + `flagcol_offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "black") +
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = (`flagcol_offset_model_regression_estimate_(Intercept)` - `flagcol_offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`flagcol_offset_model_regression_estimate_.$log_Radius_um` - `flagcol_offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "black") +
  geom_abline(aes(intercept = `offset_model_regression_estimate_(Intercept)`/ln_log10_conversion, 
                  slope = `offset_model_regression_estimate_.$log_Radius_um`/ln_log10_conversion, 
                  linetype = offset_regression_type), 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` + `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` + `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` - `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` - `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_text(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
            aes(x = 1.1, y = -4.7, label = paste("p-value =", signif(`flagcol_offset_model_regression_p.value_.$log_Radius_um`, digits = 2))), color = "black", check_overlap = TRUE) +
  geom_text(aes(x = 1.1, y = -4.5, label = paste("p-value =", signif(`offset_model_regression_p.value_.$log_Radius_um`, digits = 2))), color = "blue", check_overlap = TRUE) +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  scale_linetype_manual(name = "Regression type", values = c("poisson" = "solid" , "quasipoisson" = "twodash")) +
  scale_fill_manual(name = "Phylum", values =TaxaColors, guide = "none") +
  coord_cartesian(ylim = c(ROSMin, ROSMax)) +
  facet_grid(cols = vars(SupOx), rows = vars(Includes)) +
  scale_shape_manual(values = c(21,19,23,18)) +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text = element_text(size=11),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=11), 
        legend.title = element_text(size=13),
        strip.text = element_text(size=11)) +
  labs(x = ~log[10] ~ "(Median Cell Radius (µm))", 
       y = ~log[10] ~ "(Genes Encoding "~O[2]^`•-` ~" Metabolizing Enzymes Normalized to Total Genes)")

# ggsave(paste("SupOxRadPlots",".png",sep = ""), plot = SupOxRadPlots, device = NULL, path = Figures,scale = 1, height=7, width= 10, units = c("in"),dpi = 300, limitsize = TRUE)
ggsave(paste("Figure3",".png",sep = ""), plot = SupOxRadPlots, device = NULL, path = Figures,scale = 1, height=7, width= 10, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r SupOxRadPlots, fig.cap = SupOxRadPlots_cap, fig.height= 7, fig.width= 7, echo = FALSE}
SupOxRadPlots_cap <- glue("**Comparison of log~10~ (Total number of genes encoding O~2~^•−^ metabolizing enzymes ('SupOx_count') normalized to the total number of genes present in each organism ('GeneModels_count')) vs. the log~10~ (median cell radius in µm ('log_Radius_um')).**", 
                            "Poisson (solid line) or Quasi-Poisson (dashed line) regressions fitted to data ± Standard Error (dotted line).",
                            "Regressions were run with (black line) or without (blue line) 'Colony' and 'Flagella' as co-variates.", 
                            "Selected prokaryote genomes are presented for comparison, but excluded from the presented regressions.",
                            "Symbol color corresponds to taxon lineage ('Phylum').", 
                            # "Citations for data sources can be found in Supplementary Table S3.", 
                          .sep = " ")
SupOxRadPlots
```

With increasing cell radius, eukaryotic phytoplankton have a smaller fraction of their total genes that scavenge O~2~^•−^, as shown by the negative slope of the `r SupOxRad$offset_regression_type` regression (Figure \@ref(fig:SupOxRadPlots), Blue line, Slope = `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck('offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck('offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p = `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck('offset_model_regression', 1, "p.value", 2))`). The negative slope does not support our Hypothesis \@ref(hyp:SupOx) that phytoplankton do not differentially allocate a changing fraction of their total gene content to O~2~^•−^ production nor scavenging. 
Including 'Flagella' and 'Colony' as co-variates in the regression results, however, in a slope that is not statistically different from zero (Figure \@ref(fig:SupOxRadPlots), Black line, Slope = `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck('flagcol_offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck('flagcol_offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p = `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck('flagcol_offset_model_regression', 1, "p.value", 2))`), driven by the influence of 'Flagella' (p-value = `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck("flagcol_offset_model_pvalue", 1, tidy, "p.value", 3))`) but not 'Colony' (p-value = `r ScientificTen(SupOxRad %>% filter(SupOx == "Scavenging") %>% pluck("flagcol_offset_model_pvalue", 1, tidy, "p.value", 4))`). Including data from selected prokaryotic phytoplankton did not qualitatively alter these results. O~2~^•−^ metabolism in phytoplankton appears to be mediated by a nearly fixed set of core genes that do not change with increasing gene count, thus the fractional gene allocation to O~2~^•−^ decreases as cell radius (and the co-varying total gene count) increases. 

XXX further reiterate the influence of the flagella to address reviewer #1's comments. The influence of 'Flagella' likely drives the 

### Influences of Flagella, Colony Formation and Cell Shape on Gene Counts for Enzymes Metabolizing O~2~^•−^ {.unnumbered}

```{r SupOxFlag_pval, include = FALSE}
SupOxFlag_pval <- MergedData %>%
  select(FileName, Flagella, SupOx_count, GeneModels_count, Phylum, SupOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(SupOx == "Production" | SupOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(Flagella), 
         !is.na(SupOx),
         Taxa != "Prokaryote") %>%
  unique() %>%
  mutate(norm_SupOx = SupOx_count/GeneModels_count) %>%
  compare_means(norm_SupOx ~ Flagella, data = ., method = "wilcox.test")
```

```{r generate SupOxFlagPlot, include = FALSE}
SupOxFlagPlot <- MergedData %>%
  filter(SupOx == "Production" | SupOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(Flagella), 
         !is.na(SupOx),
         Taxa != "Prokaryote") %>% #Taxa != "Prokaryote"
  select(FileName, Flagella, SupOx_count, GeneModels_count, Phylum, SupOx, Genus) %>%
  unique() %>%
  ggplot(aes(x = Flagella, y = SupOx_count/GeneModels_count)) +
  geom_boxplot(notch = TRUE, outlier.shape = NA) +
  geom_jitter(aes(color = Phylum),width = 0.2, stat = "unique") +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  xlab("") +
  stat_compare_means(comparisons = MyFlagellaComparisons, method = "wilcox.test") +
  facet_grid(cols = vars(c(SupOx))) +
  scale_x_discrete(labels = c("no" = "No Flagella", "yes" = "Flagella")) +
  scale_y_continuous(labels = scales::scientific)  +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_text(size=9),
        axis.text.x = element_text(size=12),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=10), 
        legend.title = element_text(size=10),
        strip.text = element_text(size=11)) +
  labs(y = "Genes Encoding "~O[2]^`•-` ~" Scavenging Enzymes Normalized to Total Genes")

# ggsave(paste("HyPePennateCentricPlot",".png",sep = ""), plot = HyPePennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

ggsave(paste("Figure4",".png",sep = ""), plot = SupOxFlagPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r SupOxFlagPlot, fig.cap = SupOxFlagPlot_cap, fig.height= 7, fig.width= 7, echo = FALSE}
SupOxFlagPlot_cap <- glue("**Comparison of total number of genes encoding O~2~^•−^ scavenging enzymes ('SupOx_count') normalized to the total number of genes present in each organism ('GeneModels_count')) vs. the presence or absence of flagella in the organism.**",
                          "Symbol color corresponds to taxon lineage ('Phylum').",
                          "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                          "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
SupOxFlagPlot
```

Consistent with the significant influence of flagella on the regressions vs median cell radius (Figure \@ref(fig:SupOxRadPlots)), flagellated phytoplankton have a smaller proportion of their total gene content encoding O~2~^•−^ scavenging (Figure \@ref(fig:SupOxFlagPlot), p = `r ScientificTen(SupOxFlag_pval$p)`), than do non-flagellated phytoplankton. This suggests that cellular motility contributes to phytoplankton homeostasis of O~2~^•−^.

```{r SupOxColony_pval, include = FALSE}
SupOxColony_pval <- MergedData %>%
  select(FileName, Flagella, SupOx_count, GeneModels_count, Phylum, SupOx, Genus, Taxa, ColonySpecies) %>%
  filter(SupOx == "Scavenging",
         !is.na(ColonySpecies),
         GeneModels_count != 0, 
         Taxa != "Prokaryote") %>%
  mutate(norm_SupOx = SupOx_count/GeneModels_count) %>%  
  unique() %>%
  compare_means(norm_SupOx ~ ColonySpecies, data = ., method = "wilcox.test")
```

```{r generate SupOxColonyPlot, include = FALSE}
# SupOxColonyPlot <- MergedData %>%
#   filter(SupOx == "Production" | SupOx == "Scavenging",
#          GeneModels_count != 0,
#          !is.na(ColonySpecies),
#          Taxa != "Prokaryote") %>% #Taxa != "Prokaryote"
#   select(FileName, ColonySpecies, SupOx_count, GeneModels_count, Phylum, SupOx, Genus) %>%
#   unique() %>%
#   ggplot(aes(x = ColonySpecies, y = SupOx_count/GeneModels_count)) +
#   geom_boxplot(notch = TRUE, outlier.shape = NA) +
#   geom_jitter(aes(color = Phylum),width = 0.2, stat = "unique") +
#   theme_bw() +
#   theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
#   xlab("") +
#   stat_compare_means(comparisons = MyColonyComparisons, method = "wilcox.test") +
#   facet_grid(cols = vars(c(SupOx))) +
#   scale_x_discrete(labels = c("FALSE" = "Non-colony", "TRUE" = "Colony")) +
#   scale_y_continuous(labels = scales::scientific)  +
#   scale_colour_manual(name = "Phylum", values =TaxaColors) +
#   theme(panel.grid.major = element_blank(),
#         panel.grid.minor = element_blank(),
#         axis.text.y = element_text(size=9),
#         axis.text.x = element_text(size=12),
#         axis.title = element_text(size=12),
#         legend.text = element_text(size=10),
#         legend.title = element_text(size=10),
#         strip.text = element_text(size=11))  +
#   labs(y = "Genes Encoding "~O[2]^`•-` ~" Metabolizing Enzymes Normalized to Total Genes")

# ggsave(paste("HyPePennateCentricPlot",".png",sep = ""), plot = HyPePennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r SupOxColonyPlot, fig.cap = SupOxColonyPlot_cap, fig.height= 7, fig.width= 3.5, echo = FALSE}
SupOxColonyPlot_cap <- glue("**Comparison of total number of genes encoding O~2~^•−^ scavenging enzymes ('SupOx_count') normalized to the total number of genes present in each organism ('GeneModels_count') vs. the ability of the organism to form colonies.**",
                            "Symbol color corresponds to taxon lineage ('Phylum').",
                            "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                            "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
# SupOxColonyPlot
```

Setting aside any influence of cell size, colony and non-colony forming phytoplankton do not significantly differ in the fraction of their total gene content encoding O~2~^•−^ scavenging (p = `r ScientificTen(SupOxColony_pval$p)`) (Data not presented).

```{r SupOxDiatomShape_pval, include = FALSE}
SupOxScavPennateCentricPlot_pval <- MergedData %>%
  select(FileName, Flagella, SupOx_count, GeneModels_count, Phylum, SupOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(Taxa == "Diatom",
         SupOx == "Scavenging",
         GeneModels_count != 0) %>%
  unique() %>%
  mutate(norm_SupOx = SupOx_count/GeneModels_count) %>%
  compare_means(norm_SupOx ~ PennateCentric, data = ., method = "wilcox.test")
```


```{r generate SupOxPennateCentricPlot, include = FALSE}
# SupOxPennateCentricPlot <- MergedData %>%
#   filter(Taxa == "Diatom",
#          SupOx == "Scavenging",
#          GeneModels_count != 0) %>%
#   select(FileName, PennateCentric, SupOx_count, GeneModels_count, Phylum, SupOx, Genus) %>%
#   unique() %>%
#   ggplot(aes(x = PennateCentric, y = SupOx_count/GeneModels_count)) +
#   geom_boxplot(notch = TRUE, outlier.shape = NA) +
#   geom_jitter(aes(colour = Genus),width = 0.2, stat = "unique") +
#   # coord_cartesian(ylim = c(ROSBoxPlotMin, ROSBoxPlotMax)) +
#   theme_bw() +
#   theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
#   # ylab("Total number of Genes encoding Superoxide Scavenging") +
#   xlab("") +
#   # ggtitle("SupOx Scavenging") +
#   facet_grid(cols = vars(SupOx)) +
#   stat_compare_means(comparisons = MyShapeComparisons, method = "wilcox.test") +
#   # ggforce::facet_zoom(ylim = c(0, ROSBoxPlotMax)) +
#   scale_y_continuous(labels = scales::scientific) +
#   theme(panel.grid.major = element_blank(),
#         panel.grid.minor = element_blank(),
#         axis.text.y = element_text(size=9),
#         axis.text.x = element_text(size=12),
#         axis.title = element_text(size=12), 
#         legend.text = element_text(size=10), 
#         legend.title = element_text(size=10),
#         strip.text = element_text(size=11)) +
#   labs(y = "Genes Encoding "~O[2]^`•-` ~" Metabolizing Enzymes Normalized to Total Genes")

# ggsave(paste("SupOxPennateCentricPlot",".png",sep = ""), plot = SupOxPennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

```

```{r SupOxPennateCentricPlot, fig.cap = SupOxPennateCentricPlot_cap, fig.height= 3.5, fig.width= 3.5, echo = FALSE}
SupOxPennateCentricPlot_cap <- glue("**Comparison of total number of genes encoding O~2~^•−^ scavenging enzymes ('SupOx_count') normalized to the total number of genes present in each diatom ('GeneModels_count') vs. the growth form of the diatom ('PennateCentric').**",
                                      "Symbol color corresponds to the genus of the diatom ('Genus').", 
                                    "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                                      "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
# SupOxPennateCentricPlot
```

Pennate and centric diatoms O~2~^•−^ have similar fractions of their genomes encoding O~2~^•−^ scavenging (p = `r ScientificTen(SupOxScavPennateCentricPlot_pval$p)`) (Data not presented).
Our results support our hypothesis that differential diffusional exchange across diatoms of different shape does not influence the fraction of total gene content that encodes O~2~^•−^ scavenging enzymes, because O~2~^•−^ diffusion is blocked by the cell membrane irrespective of cell shape. XXX supporting Hypothesis \@ref(hyp:SupOx) 
Given that the O~2~^•−^ is poorly diffusible across membranes, intracellularly produced O~2~^•−^ has to be scavenged to limit detrimental reactions of O~2~^•−^ [@bergaminiOxygenReactiveOxygen2004]. As a result, cells universally maintain the genomic capacity to scavenge O~2~^•−^. Differences between genomic patterns of pennate and centric diatoms may arise when comparing metallo-forms of SOD, noting that @groussmanDiversityEvolutionaryHistory2015 found that pennate diatoms transcribe Cu/Zn-SOD but not Fe-SOD, whereas centric diatoms transcribe Fe-SOD more frequently than they transcribe Cu/Zn-SOD.

## Hydrogen Peroxide {.unnumbered}

```{r read in HyPe models, include = FALSE}
HyPeRad <- readRDS(file.path(DataProcessedModels, "HyPeRad.Rds"))
OchrophyteHyPeRad <- readRDS(file.path(DataProcessedModels, "OchrophyteHyPeRad.Rds"))
HyPeRadProk <- readRDS(file.path(DataProcessedModels, "HyPeRadProk.Rds"))
# HyPeLat <- readRDS(file.path(DataProcessedModels, "HyPeLat.Rds"))
# HyPeLatProk <- readRDS(file.path(DataProcessedModels, "HyPeLatProk.Rds"))
HyPeMerged <- readRDS(file.path(DataProcessed, "HyPeMerged.Rds"))
```

All prokaryotic (Figure \@ref(fig:ProkHyPeDotPlot)) and eukaryotic (Figure \@ref(fig:EukHyPeDotPlot)) phytoplankton (with the exception of a single transcriptome from the prasinophyte *Micromonas polaris*) have genes encoding H~2~O~2~ producing enzymes, as they all carry gene(s) encoding the ubiquitous enzyme Superoxide Dismutase. Three other oxidases are also very well conserved; Copropophyrinogen oxidase is found across all eukaryotic and prokaryotic phytoplankton, with the exception of one transcriptome. Thiol oxidase and acyl CoA oxidase are found in nearly all eukaryotic phytoplankton, with the exception of three transcriptomes. Other noticeable taxonomic differences include L-aspartate oxidase is found in nearly all prokaryotes, all green algae, but hardly any other 'Taxa'. Sarcosine oxidase is not present in small diatoms, and small green algae, but is present in nearly all diatoms and haptophytes. (S)-2-hydroxy-acid oxidase is only present in 3 of the 24 *Prochlorococcus* species analyzed, but is present in all other prokaryotes, with the exception of *Synechococcus sp.* MITS9220. 

All eukaryotic (Figure \@ref(fig:EukHyPeDotPlot)) and prokaryotic phytoplankton excluding a few from the *Prochlorococcus* and *Synechococcus* picocyanobacterial genera (Figure \@ref(fig:ProkHyPeDotPlot)), have genes encoding H~2~O~2~ scavenging enzymes. Some strains of *Prochlorococcus* and *Synechococcus* have lost the genomic capacity to scavenge H~2~O~2~, and would rely on co-occuring hosts for H~2~O~2~ scavenging XXX citation. Similar patterns are also seen in pico-eukaryotes such as the prasinophytes, whom while maintaining the genomic capacity to scavenge ROS, have lost certain enzymes such as catalase but maintain the genomic capacity to scavenge H~2~O~2~ using enzymes like ascorbate peroxidase, glutathione peroxidase and Cytochrome C peroxidase (Figure \@ref(fig:EukHyPeDotPlot)). 

The absence from catalase from most cyanobacterial genomes is not surprising. @bernroitnerOccurrencePhylogenyStructure2009 analyzed 44 different cyanobacterial genomes and found that only *Nostoc punctiforme* PCC73102 conserved the full catalase encoding gene, whereas *Synechococcus elongatus* PCC7942 and *Cyanothece sp.* ATCC 51142 maintained a partial catalase encoding gene. In our analyses, only *Synechococcus elongatus* PCC11802 maintained a catalase encoding gene (Figure \@ref(fig:ProkHyPeDotPlot)). In the greens, catalase has been lost from the smaller greens but is maintained in the larger greens (Figure \@ref(fig:EukHyPeDotPlot)). The loss of catalase in smaller green algae is evidence of the Black Queen Hypothesis in action [@morrisBlackQueenHypothesis2012], in that H~2~O~2~ can passively diffuse out of the smaller green algae but cannot diffuse out of larger green algae. Loss of function mutations in catalase encoding genes in small algae are therefore less deleterious than they would be to large green algae. It is also worth questioning whether catalase is not very well conserved because the K~M~ of the peroxidase enzymes is in the low µM range, as opposed to a K~M~ of ~220 mM for catalases [@bernroitnerOccurrencePhylogenyStructure2009]. 

When looking at the prokaryotic phytoplankton, *Prochlorococcus* does not maintain the genomic capacity to scavenge H~2~O~2~, and as would have outsourced this task to other members of the community [@morrisBlackQueenHypothesis2012]. Among prokaryotic phytoplankton, our results support the earlier esuggestion that increased genomic capacity for H~2~O~2~ scavenging in *Synechococcus* compared to *Prochlorococcus* is a result of the larger size in *Synechococcus* compared to *Prochlorococcus* [@morrisDependenceCyanobacteriumProchlorococcus2011] (Figure \@ref(fig:ProkHyPeDotPlot)). It is however important to note the vast differences between prokaryotic and eukaryotic phytoplankton, with most eukaryotic phytoplankton, irrespective of lineage, maintaining the genomic capacity to produce ascorbate peroxidase, glutathione peroxidase and Cytochrome C peroxidase (Figure \@ref(fig:EukHyPeDotPlot)). Peroxidases are often involved in pathways beyond simple ROS scavenging, including the Halliwell-Asada cycle where ascorbate peroxidase is involved [@pandeyRedoxHomeostasisGene2015]. *Ostreococcus*, the smallest prasinophyte has a radius of 0.5 µm, similar to that of the prokaryote *Synechococcus* (Table \@ref(tab:GenomeTable)), and would therefore share a similar diffusion path length. Nevertheless *Ostreococcus*, in common with other eukaryotes, retains the genomic capacities to produce ascorbate peroxidase, glutathione peroxidase and Cytochrome C peroxidase, which may thus reflect the "cost of being eukaryotic" (Figure \@ref(fig:EukHyPeDotPlot)). XXX to be dealt with by co-occuring cells (cite morris 2012).

### Influence of Cell Size on Gene Counts for Enzymes Metabolizing H~2~O~2~ {.unnumbered}

```{r, include = FALSE}
HyPeDiff <- MergedData %>%
  filter(Taxa != "Prokaryote",
         !is.na(HyPe)) %>%
  select(FileName, Genus, species, Strain, Name, Ome, Taxa, Rad1_um, Rad2_um, Rad3_um, Flagella, GenomeSize_mbp, GeneModels_count, Latitude, Longitude, Marine, HyPe, PennateCentric, SA_um2, Volume_um3, SAVol_um, Radius_um, log_Radius_um, log_GenomeSize_mbp, log_GeneModels_count, abs_Latitude, log_Volume_um3, log_SA_um2, log_SAVol_um, HyPe_count, ColonySpecies, Phylum) %>%
  unique() %>%
  filter(GeneModels_count != 0) %>% #This is because of introducing exposure
  glm(.$HyPe_count ~ .$log_Radius_um + HyPe , offset = log(GeneModels_count), data = ., family = quasipoisson(link = "log"))

HyPeDiff2 <- MergedData %>%
  filter(Taxa != "Prokaryote",
         !is.na(HyPe)) %>%
  select(FileName, Genus, species, Strain, Name, Ome, Taxa, Rad1_um, Rad2_um, Rad3_um, Flagella, GenomeSize_mbp, GeneModels_count, Latitude, Longitude, Marine, HyPe, PennateCentric, SA_um2, Volume_um3, SAVol_um, Radius_um, log_Radius_um, log_GenomeSize_mbp, log_GeneModels_count, abs_Latitude, log_Volume_um3, log_SA_um2, log_SAVol_um, HyPe_count, ColonySpecies, Phylum) %>%
  unique() %>%
  filter(GeneModels_count != 0) %>% #This is because of introducing exposure
  glm(.$HyPe_count ~ .$log_Radius_um * HyPe , offset = log(GeneModels_count), data = ., family = quasipoisson(link = "log"))

anova(HyPeDiff, HyPeDiff2, test = "F")

emmeans::emmeans(HyPeDiff, "HyPe")
HyPeDiff_pvalue <- pairs(emmeans::emmeans(HyPeDiff, "HyPe")) %>% tidy()
HyPeDiff %>% anova(test = "F")
```

```{r generate HyPeRadPlots, include = FALSE}
HyPeRad$Includes <- c("Eukaryote Regressions")
HyPeRadProk$Includes <- c("Eukaryotes + Prokaryotes")
# DiatomHyPeRad$Includes <- c("Diatom")


OchrophyteHyPeRad <- OchrophyteHyPeRad %>% 
  mutate(ROSType = "HyPe",
         ROSName = "H~2~O~2~") %>%
  rename(ROSMetabolism = HyPe)

HyPeRadPlots <- rbind(HyPeRad, HyPeRadProk) %>%
  filter(HyPe == "Production" | HyPe == "Scavenging") %>%
  unnest(c(offset_model_regression), names_sep = "_") %>%
  unnest(c(flagcol_offset_model_regression), names_sep = "_") %>%
  pivot_wider(., names_from = c(flagcol_offset_model_regression_term), values_from = c(flagcol_offset_model_regression_estimate, flagcol_offset_model_regression_std.error, flagcol_offset_model_regression_statistic, flagcol_offset_model_regression_p.value)) %>%
  pivot_wider(., names_from = c(offset_model_regression_term), values_from = c(offset_model_regression_estimate, 
                                                                               offset_model_regression_std.error, offset_model_regression_statistic, offset_model_regression_p.value)) %>%
  unnest(c(offset_model_pvalue), names_sep = "_") %>%
  unnest(c(flagcol_offset_model_pvalue), names_sep = "_") %>%
  unnest(cols = c(data)) %>%
  filter(Includes != "Eukaryotes + Prokaryotes") %>%
  unique() %>%
  ggplot() +
  geom_point(aes(x = log_Radius_um, y = log10(HyPe_count/GeneModels_count), color = Phylum)) +
  geom_point(aes(x = log_Radius_um, y = log10(HyPe_count/GeneModels_count), color = Phylum), data = MergedData %>% filter(Phylum == "Prokaryote", !is.na(HyPe))) + 
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = `flagcol_offset_model_regression_estimate_(Intercept)`/ln_log10_conversion, 
                  slope = `flagcol_offset_model_regression_estimate_.$log_Radius_um`/ln_log10_conversion, 
                  linetype = flagcol_offset_regression_type), 
              color = "black") +
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = (`flagcol_offset_model_regression_estimate_(Intercept)` + `flagcol_offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`flagcol_offset_model_regression_estimate_.$log_Radius_um` + `flagcol_offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "black") +
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = (`flagcol_offset_model_regression_estimate_(Intercept)` - `flagcol_offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`flagcol_offset_model_regression_estimate_.$log_Radius_um` - `flagcol_offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "black") +
  geom_abline(aes(intercept = `offset_model_regression_estimate_(Intercept)`/ln_log10_conversion, 
                  slope = `offset_model_regression_estimate_.$log_Radius_um`/ln_log10_conversion, 
                  linetype = offset_regression_type), 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` + `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` + `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` - `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` - `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_text(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
            aes(x = 1.1, y = -4.1, label = paste("p-value =", signif(`offset_model_regression_p.value_.$log_Radius_um`, digits = 2))), color = "blue", check_overlap = TRUE) +
  geom_text(aes(x = 1.1, y = -4.3, label = paste("p-value =", (signif(`flagcol_offset_model_regression_p.value_.$log_Radius_um`, digits = 2)))), color = "black", check_overlap = TRUE) +
  # geom_point(data = MergedData %>% filter(Taxa == "Prokaryote", HyPe == "Production" | HyPe == "Scavenging"), aes(x = log_Radius_um, y= log10((HyPe_count/GeneModels_count)), color = Phylum)) +
  # ggalt::geom_encircle(data = MergedData %>% filter(Taxa == "Prokaryote", HyPe == "Production" | HyPe == "Scavenging"), alpha = 0.1, aes(x=log_Radius_um, y= log10((HyPe_count/GeneModels_count)), color = Phylum), expand = 0, s_shape = 0.9) +
  facet_grid(cols = vars(HyPe), rows = vars(Includes)) +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  # gfun::scale_color_manual2(name = "Contains", values = c("blue", "black")) +
  scale_linetype_manual(name = "Regression type", values = c("poisson" = "solid" , "quasipoisson" = "twodash")) +
  scale_shape_manual(values = c(21,19,23,18)) +
  coord_cartesian(ylim = c(ROSMin, ROSMax)) +
  # scale_x_continuous(limits = c(-0.5, 1.6)) +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text = element_text(size=11),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=11), 
        legend.title = element_text(size=13),
        strip.text = element_text(size=11)) +
  labs(x = ~log[10] ~ "(Median Cell Radius (µm))", 
       y = ~log[10] ~ "(Genes Encoding "~H[2]~O[2]~" Metabolizing Enzymes Normalized to Total Genes)") 

# ggsave(paste("HyPeRadPlots",".png",sep = ""), plot = HyPeRadPlots, device = NULL, path = Figures,scale = 1, height=7, width= 10, units = c("in"),dpi = 300, limitsize = TRUE)

ggsave(paste("Figure5",".png",sep = ""), plot = HyPeRadPlots, device = NULL, path = Figures,scale = 1, height=7, width= 10, units = c("in"),dpi = 300, limitsize = TRUE)

```

```{r HyPeRadPlots, fig.cap = HyPeRadPlots_cap, fig.height= 7, fig.width= 10, echo = FALSE}
HyPeRadPlots_cap <- glue("**Comparison of log~10~ (Total number of genes encoding H~2~O~2~ metabolizing enzymes ('HyPe_count') normalized to the total number of genes present in each organism ('GeneModels_count')) vs. the log~10~( median cell radius in µm ('log_Radius_um')).**",
                            "Poisson (solid line) or Quasi-Poisson (dashed line) regressions fitted to data ± Standard Error (dotted line).",
                            "Regressions were run with (black line) or without (blue line) 'Colony' and 'Flagella' as co-variates.", 
                            "Selected prokaryote genomes are presented for comparison, but excluded from the presented regressions.", 
                            "Symbol color corresponds to taxon lineage ('Phylum').",
                            "Citations for data sources are in Supplementary Table S3. ",
                         .sep = " ")

HyPeRadPlots
```

With increasing cell radius, eukaryotic phytoplankton have a smaller fraction of their total genes that encode the production of H~2~O~2~, as shown by the negative slope of the `r HyPeRad$offset_regression_type[[1]]` regression (Figure \@ref(fig:HyPeRadPlots), Blue line, Slope = `r ScientificTen(HyPeRad %>% filter(HyPe == "Production") %>% pluck('offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(HyPeRad %>% filter(HyPe == "Production") %>% pluck('offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p = `r ScientificTen(HyPeRad %>% filter(HyPe == "Production") %>% pluck('offset_model_regression', 1, "p.value", 2))`). Including 'Flagella' and 'Colony' as co-variates did not influence the slope of the fraction of total genes encoding H~2~O~2~ production with increasing median cell radius (Black line, 'Flagella' p-value = `r ScientificTen(HyPeRad$flagcol_offset_model_pvalue[[1]] %>% pluck("p.value",3))`, 'Colony' p-value = `r ScientificTen(HyPeRad$flagcol_offset_model_pvalue[[1]] %>% pluck("p.value",4))`). Including data from selected prokaryotic phytoplankton did not qualitatively alter these results. The pattern of a smaller fraction of total genes for H~2~O~2~ production with increasing cell radius supports our Hypothesis \@ref(hyp:HyPeNitOxRad) that large phytoplankton counter decreasing diffusional loss of H~2~O~2~ out of cells by having lower genomic capacity for H~2~O~2~ production, while losses of genes encoding H~2~O~2~ producing enzymes is more costly to small phytoplankton (Figure \@ref(fig:HyPeRadPlots)).  @diazProductionExtracellularReactive2018 found that a major factor that influences  the capacity for production of H~2~O~2~ is whether or not the organism can form blooms, in that bloom forming species produce more H~2~O~2~. The ability to form blooms was not analyzed in our data. 

With increasing cell radius, eukaryotic phytoplankton have a smaller fraction of their total genes that encode the capacity to scavenge H~2~O~2~, as shown by the negative slope of the `r HyPeRad$offset_regression_type[[2]]` regression (Figure \@ref(fig:HyPeRadPlots), Blue line, Slope = `r ScientificTen(HyPeRad %>% filter(HyPe == "Scavenging") %>% pluck('offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(HyPeRad %>% filter(HyPe == "Scavenging") %>% pluck('offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p = `r ScientificTen(HyPeRad %>% filter(HyPe == "Scavenging") %>% pluck('offset_model_regression', 1, "p.value", 2))`). Including 'Flagella' and 'Colony' as co-variates did not influence the slope of the fraction of total genes encoding H~2~O~2~ scavenging with increasing median cell radius (Figure \@ref(fig:HyPeRadPlots), Black line, 'Flagella' p-value = `r ScientificTen(HyPeRad$flagcol_offset_model_pvalue[[2]] %>% pluck("p.value",3))`, 'Colony' p-value = `r ScientificTen(HyPeRad$flagcol_offset_model_pvalue[[2]] %>% pluck("p.value",4))`). Including data from selected prokaryotic phytoplankton did not qualitatively alter these results. A similar study solely focusing on small phytoplankton such as pico-cyanobacteria and pico-prasinophytes might yield different results as more genomes are sequenced, especially considering that @randhawaApplicabilityHydrogenPeroxide2012 found that H2O2 added to seawater at a concentration of 1.6 mg L^−1^ did not affect cells with a radius larger than 1 to 1.5 μm, and mainly harmed the prasinophyte *Micromonas pusilla*.

Noting that the fraction of total genes allocated to both H~2~O~2~ production and scavenging decrease with increasing median cell radius, we found that the fraction of total genes that encode production of H~2~O~2~ decreases proportionally more with increasing cell radius than does the fraction of total genes that encode scavenging of H~2~O~2~ (`r ScientificTen(HyPeDiff_pvalue$p.value)`). The difference in slopes again supports our Hypothesis \@ref(hyp:HyPeNitOxRad) that large phytoplankton counter decreasing diffusional loss of H~2~O~2~ out of cells by having lower genomic capacity for H~2~O~2~ production relative to H~2~O~2~ scavenging. Because median cell radius co-varied with Phylum, we generally excluded co-variate from our regressions, in order to focus on any cross-taxon patterns with changing median cell radius. Nevertheless, representatives of the Ochrophyte Phylum alone spanned more than an order of magnitude in median cell radius. We therefore tested whether the log~10~(total number of genes encoding the metabolism of O~2~^•−^, H~2~O~2~ or ^•^NO) varied with the log~10~ (median cell radius) across the Ochrophytes alone (Figure \@ref(fig:OchrophyteRadPlot)). We found that across Ochrophytes, the fraction of total genes encoding the production of H~2~O~2~ decreased with increasing cell radius (Slope = `r ScientificTen(OchrophyteHyPeRad %>% filter(ROSMetabolism == "Production") %>% pluck('offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(OchrophyteHyPeRad %>% filter(ROSMetabolism == "Production") %>% pluck('offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`), although the p value for the regression was `r ScientificTen(OchrophyteHyPeRad %>% filter(ROSMetabolism == "Production") %>% pluck('offset_model_regression', 1, "p.value", 2))`. This marginal decrease in the total number of genes encoding H~2~O~2~ production with increasing median cell radius in ochrophytes again tends to support our Hypothesis \@ref(hyp:HyPeNitOxRad), with data from within a single phylum.

### Influences of Flagella, Colony Formation and Cell Shape on Gene Counts for Enzymes Metabolizing H~2~O~2~ {.unnumbered}

```{r HyPeFlagPlot_pval, include = FALSE}

HyPeFlagellaDelta <- MergedData %>%
  filter(HyPe == "Production" | HyPe == "Scavenging",
         GeneModels_count != 0,
         !is.na(Flagella),
         Taxa != "Prokaryote") %>% 
  select(FileName, HyPe, HyPe_count, GeneModels_count, Flagella) %>%
  unique() %>%
  pivot_wider(names_from = HyPe, values_from = HyPe_count) %>%
  mutate(prodNorm = Production/GeneModels_count,
         scavNorm = Scavenging/GeneModels_count) %>%
  group_by(Flagella) %>%
  summarize(Production = median(prodNorm), 
            Scavenging = median(scavNorm)) %>%
  mutate(deltaProduction = (Production[2] - Production[1])/ Production[1],
         deltaScavenging = (Scavenging[2] - Scavenging[1])/ Scavenging[1])
  

HyPeProdFlagPlot_pval <- MergedData %>%
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(HyPe == "Production",
         !is.na(Flagella), 
         GeneModels_count != 0,
         Phylum != "Prokaryote") %>%
  mutate(norm_HyPe = HyPe_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_HyPe ~ Flagella, data = ., method = "wilcox.test")

HyPeScavFlagPlot_pval <- MergedData %>%
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus) %>%
  filter(HyPe == "Scavenging",
         !is.na(Flagella), 
         GeneModels_count != 0,
         Phylum != "Prokaryote") %>%
  mutate(norm_HyPe = HyPe_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_HyPe ~ Flagella, data = ., method = "wilcox.test")
```
 
```{r generate HyPeFlagPlot, include = FALSE}
HyPeFlagPlot <- MergedData %>%
  filter(HyPe == "Production" | HyPe == "Scavenging",
         GeneModels_count != 0,
         !is.na(Flagella), 
         Taxa != "Prokaryote") %>% #Taxa != "Prokaryote"
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus) %>%  
  unique() %>%
  ggplot(aes(x = Flagella, y = HyPe_count/GeneModels_count)) +
  geom_boxplot(notch = TRUE, outlier.shape = NA) +
  geom_jitter(aes(color = Phylum),width = 0.2, stat = "unique") +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  xlab("") +
  stat_compare_means(comparisons = MyFlagellaComparisons, method = "wilcox.test") +
  facet_grid(cols = vars(c(HyPe))) +
  scale_x_discrete(labels = c("no" = "No Flagella", "yes" = "Flagella")) +
  scale_y_continuous(labels = scales::scientific)  +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_text(size=9),
        axis.text.x = element_text(size=12),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=10), 
        legend.title = element_text(size=10),
        strip.text = element_text(size=11)) +
  labs(y = "Genes Encoding "~H[2]~O[2]~" Metabolizing Enzymes Normalized to Total Genes")

ggsave(paste("Figure6",".png",sep = ""), plot = HyPeFlagPlot, device = NULL, path = Figures,scale = 1, height=7, width= 10, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r HyPeFlagPlot, fig.cap = HyPeFlagPlot_cap, fig.height = 7, fig.width = 7, echo = FALSE}
HyPeFlagPlot_cap <- glue("**Comparison of total number of genes encoding H~2~O~2~ metabolizing enzymes ('HyPe_count') normalized to the total number of genes present in each organism ('GeneModels_count') vs. the presence or absence of flagella in the organism.**", 
                                       "Symbol color corresponds to taxon lineage ('Phylum').", 
                         "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                                       "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
HyPeFlagPlot
```

Setting aside the regressions vs median cell radius, non-flagellated vs. flagellated phytoplankton exhibit no statistically significant difference in the fraction of their total genes encoding the production of H~2~O~2~ (p = `r ScientificTen(HyPeProdFlagPlot_pval$p)`), whereas non-flagellated phytoplankton have a significantly larger proportion of their total gene content encoding H~2~O~2~ scavenging (p = `r ScientificTen(HyPeScavFlagPlot_pval$p)`), than do flagellated phytoplankton (Figure \@ref(fig:HyPeFlagPlot)). Thus, presence of flagella may contribute to the maintenance of H~2~O~2~ homeostasis across eukaryotic phytoplankton, but this potential influence is obscured by the range of cell size across eukaryotic phytoplankton.

```{r HyPeColonyPlot_pval, include = FALSE}

HyPeColonyDelta <- MergedData %>%
  filter(HyPe == "Production" | HyPe == "Scavenging",
         GeneModels_count != 0,
         !is.na(ColonySpecies),
         Taxa != "Prokaryote") %>% 
  select(FileName, HyPe, HyPe_count, GeneModels_count, ColonySpecies) %>%
  unique() %>%
  pivot_wider(names_from = HyPe, values_from = HyPe_count) %>%
  mutate(prodNorm = Production/GeneModels_count,
         scavNorm = Scavenging/GeneModels_count) %>%
  group_by(ColonySpecies) %>%
  summarize(Production = median(prodNorm), 
            Scavenging = median(scavNorm)) %>%
  mutate(deltaProduction = (Production[2] - Production[1])/ Production[1],
         deltaScavenging = (Scavenging[2] - Scavenging[1])/ Scavenging[1])
  

HyPeProdColonyPlot_pval <- MergedData %>%
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(HyPe == "Production",
         GeneModels_count != 0,
         !is.na(ColonySpecies), 
         Taxa != "Prokaryote") %>%
  mutate(norm_HyPe = HyPe_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_HyPe ~ ColonySpecies, data = ., method = "wilcox.test")

HyPeScavColonyPlot_pval <- MergedData %>%
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(HyPe == "Scavenging",
         GeneModels_count != 0,
         !is.na(ColonySpecies), 
         Taxa != "Prokaryote") %>%
  mutate(norm_HyPe = HyPe_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_HyPe ~ ColonySpecies, data = ., method = "wilcox.test")
```

```{r generate HyPeColonyPlot, include = FALSE}
HyPeColonyPlot <- MergedData %>%
  filter(HyPe == "Production" | HyPe == "Scavenging",
         GeneModels_count != 0,
         !is.na(ColonySpecies), 
         Taxa != "Prokaryote") %>% #Taxa != "Prokaryote"
  select(FileName, ColonySpecies, HyPe_count, GeneModels_count, Phylum, HyPe, Genus) %>%
  unique() %>%
  ggplot(aes(x = ColonySpecies, y = HyPe_count/GeneModels_count)) +
  geom_boxplot(notch = TRUE, outlier.shape = NA) +
  geom_jitter(aes(color = Phylum),width = 0.2, stat = "unique") +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  xlab("") +
  stat_compare_means(comparisons = MyColonyComparisons, method = "wilcox.test") +
  facet_grid(cols = vars(c(HyPe))) +
  scale_x_discrete(labels = c("FALSE" = "Non-colony", "TRUE" = "Colony")) +
  scale_y_continuous(labels = scales::scientific)  +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_text(size=9),
        axis.text.x = element_text(size=12),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=10), 
        legend.title = element_text(size=10),
        strip.text = element_text(size=11)) +
  labs(y = "Genes Encoding "~H[2]~O[2]~" Metabolizing Enzymes Normalized to Total Genes")

# ggsave(paste("HyPePennateCentricPlot",".png",sep = ""), plot = HyPePennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

ggsave(paste("Figure7",".png",sep = ""), plot = HyPeColonyPlot, device = NULL, path = Figures,scale = 1, height=7, width= 10, units = c("in"),dpi = 300, limitsize = TRUE)

# HyPeColonyPlot
```

```{r HyPeColonyPlot, fig.cap = HyPeColonyPlot_cap, fig.height = 7, fig.width = 7, echo = FALSE}
HyPeColonyPlot_cap <- glue("**Comparison of total number of genes encoding H~2~O~2~ metabolizing enzymes ('HyPe_count') normalized to the total number of genes present in each organism ('GeneModels_count') vs. the ability of the organism to form colonies.**", 
                                       "Symbol color corresponds to taxon lineage ('Phylum').", 
                           "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                           
                                       "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
HyPeColonyPlot 
```

Setting aside any cell size influence, non-colony forming phytoplankton have a smaller proportion of their total gene content encoding both H~2~O~2~ production (p = `r ScientificTen(HyPeProdColonyPlot_pval$p)`), and also H~2~O~2~ scavenging (Figure \@ref(fig:HyPeColonyPlot), p = `r ScientificTen(HyPeScavColonyPlot_pval$p)`), than do colony forming phytoplankton. Looking at percentage change, we found that the decrease in the fraction of total genes encoding H~2~O~2~ production between non-colony and colony forming phytoplankton is smaller (`r signif(HyPeColonyDelta$deltaProduction[[1]]*100, 4)`%), than the decrease in the fraction of total genes encoding H~2~O~2~ scavenging (`r signif(HyPeColonyDelta$deltaScavenging[[1]]*100, 4)`%). Colony forming phytoplankton may have more active H~2~O~2~ metabolism with a particular emphasis on H~2~O~2~ scavenging, consistent with stronger H~2~O~2~ exchange among closely spaced cells within a colony (Hypothesis \@ref(hyp:Colony)) [@omarDiffusionalInteractionsMarine2022].

```{r HyPeDiatomShape_pval, include = FALSE}

HyPePennateCentricDelta <- MergedData %>%
  filter(HyPe == "Production" | HyPe == "Scavenging",
         Taxa == "Diatom",
         GeneModels_count != 0,
         !is.na(PennateCentric),
         Taxa != "Prokaryote") %>% 
  select(FileName, HyPe, HyPe_count, GeneModels_count, PennateCentric) %>%
  unique() %>%
  pivot_wider(names_from = HyPe, values_from = HyPe_count) %>%
  mutate(prodNorm = Production/GeneModels_count,
         scavNorm = Scavenging/GeneModels_count) %>%
  group_by(PennateCentric) %>%
  summarize(Production = median(prodNorm), 
            Scavenging = median(scavNorm)) %>%
  mutate(deltaProduction = (Production[2] - Production[1])/ Production[1],
         deltaScavenging = (Scavenging[2] - Scavenging[1])/ Scavenging[1])

HyPeProdPennateCentricPlot_pval <- MergedData %>%
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(Taxa == "Diatom",
         HyPe == "Production",
         GeneModels_count != 0,
         !is.na(PennateCentric)) %>%
  mutate(norm_HyPe = HyPe_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_HyPe ~ PennateCentric, data = ., method = "wilcox.test")

HyPeScavPennateCentricPlot_pval <- MergedData %>%
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(Taxa == "Diatom",
         HyPe == "Scavenging",
         GeneModels_count != 0,
         !is.na(PennateCentric)) %>%
  mutate(norm_HyPe = HyPe_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_HyPe ~ PennateCentric, data = ., method = "wilcox.test")
```

```{r generate HyPeProdPennateCentricPlot, include = FALSE}
HyPePennateCentricPlot <- MergedData %>%
  select(FileName, Flagella, HyPe_count, GeneModels_count, Phylum, HyPe, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(Taxa == "Diatom",
         HyPe == "Production" | HyPe == "Scavenging",
         GeneModels_count != 0,
         !is.na(PennateCentric)) %>%
  unique() %>%
  ggplot(aes(x = PennateCentric, y = HyPe_count/GeneModels_count)) +
  geom_boxplot(notch = TRUE, outlier.shape = NA) +
  geom_jitter(aes(colour = Genus),width = 0.2, stat = "unique") +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  xlab("") +
  stat_compare_means(comparisons = MyShapeComparisons, method = "wilcox.test") +
  facet_grid(cols = vars(c(HyPe))) +
  scale_y_continuous(labels = scales::scientific)  +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_text(size=9),
        axis.text.x = element_text(size=12),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=10), 
        legend.title = element_text(size=10),
        strip.text = element_text(size=11)) +
  labs(y = "Genes Encoding "~H[2]~O[2]~" Metabolizing Enzymes Normalized to Total Genes")

# ggsave(paste("HyPePennateCentricPlot",".png",sep = ""), plot = HyPePennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

```

```{r HyPePennateCentricPlot, fig.cap = HyPePennateCentricPlot_cap, fig.height = 7, fig.width = 7, echo = FALSE}
HyPePennateCentricPlot_cap <- glue("**Comparison of total number of genes encoding H~2~O~2~ metabolizing enzymes ('HyPe_count') normalized to the total number of genes in each diatom ('GeneModels_count') vs. the growth form of the diatom ('PennateCentric').**", 
                                       "Symbol color corresponds to the genus of the diatom ('Genus').", 
                                   "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                                       "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
# HyPePennateCentricPlot
```
Pennate and centric diatoms do not show statistically significant differences in the fraction of their total gene content encoding the production (p = `r ScientificTen(HyPeProdPennateCentricPlot_pval$p)`) nor the scavenging of H~2~O~2~ (p = `r ScientificTen(HyPeScavPennateCentricPlot_pval$p)`). This result does not support our Hypothesis \@ref(hyp:HyPeNitOxDiatom) that pennates have more genes encoding H~2~O~2~ producing enzymes due to their higher surface area to volume ratio (Data not presented). XXX hypothesis number

## Nitric Oxide {.unnumbered}

```{r read in NitOx Models, include = FALSE}
NitOxRad <- readRDS(file.path(DataProcessedModels, "NitOxRad.Rds"))
OchrophyteNitOxRad <- readRDS(file.path(DataProcessedModels, "OchrophyteNitOxRad.Rds"))
NitOxRadNoZero <- readRDS(file.path(DataProcessedModels, "NitOxRadNoZero.Rds"))
NitOxRadProk <- readRDS(file.path(DataProcessedModels, "NitOxRadProk.Rds"))
# NitOxLat <- readRDS(file.path(DataProcessedModels, "NitOxLat.Rds"))
# NitOxLatNoZero <- readRDS(file.path(DataProcessedModels, "NitOxLatNoZero.Rds"))
# NitOxLatProk <- readRDS(file.path(DataProcessedModels, "NitOxLatProk.Rds"))
NitOxMerged <- readRDS(file.path(DataProcessed, "NitOxMerged.Rds"))
```


In the genomes and transcriptomes that we analysed, Nitric Oxide Synthase (NOS, EC:1.14.13.39) was the most frequently occuring ^•^NO producing enzyme (Figure \@ref(fig:EukNitOxDotPlot)), but was not present or not annotated among prokaryotic phytoplankton (Data not presented). 

Nitric Oxide Dioxygenase (NOD, EC:1.14.12.17) was the most frequently occuring of the ^•^NO scavenging enzymes (Figure \@ref(fig:EukNitOxDotPlot)).
NOD sequences were identified in eukaryotes, but were either not annotated, or not present in *Prochlorococcus*, most green algae and some diatoms. NOD sequences were either not annotated or not present in *Prochlorococcus*, and were detected in *Synechococcus sp. SynAce01*, *Cyanobium sp. LEGE06143*, *Cyanobium sp. LEGE06143*, and *Crocosphaera* species.

The detection of NOS genes in our data is not surprising (Figure \@ref(fig:EukNitOxDotPlot)), considering that NOS genes have previously been detected in the prasinophyte *Ostreococcus tauri* [@foresiCharacterizationNitricOxide2010], as well as in a few pennate diatoms [@didatoTranscriptomeSequencingThree2015]. Although, while we would not detect genes encoding ^•^NO producing enzymes, a NOS-like sequence has recently been identified in *Synechococcus*, that also has Nitric Oxide Dioxygenase-like function [@piccianoNitricOxideSynthase2019]. 

XXX reconcile this paragraph with the next one.


### Influence of Cell Size on Gene Counts for Enzymes Metabolizing ^•^NO {.unnumbered}

```{r generate NitOxProdRadPlot, include = FALSE}
NitOxRad$Includes <- c("Eukaryote Regressions")
NitOxRadProk$Includes <- c("Eukaryotes + Prokaryotes")
NitOxRadNoZero$Includes <- c("Eukaryotes with zero values omitted")

NitOxRadPlots <- rbind(NitOxRad, NitOxRadProk, NitOxRadNoZero) %>%
  filter(NitOx == "Production"|NitOx == "Scavenging") %>%
  unnest(c(offset_model_regression), names_sep = "_") %>%
  unnest(c(flagcol_offset_model_regression), names_sep = "_") %>%
  # filter(flagcol_offset_model_regression_term == c('(Intercept)', '.$log_Radius_um')) %>%
  pivot_wider(., names_from = c(flagcol_offset_model_regression_term), values_from = c(flagcol_offset_model_regression_estimate, flagcol_offset_model_regression_std.error, flagcol_offset_model_regression_statistic, flagcol_offset_model_regression_p.value)) %>%
  pivot_wider(., names_from = c(offset_model_regression_term), values_from = c(offset_model_regression_estimate, offset_model_regression_std.error, offset_model_regression_statistic, offset_model_regression_p.value)) %>%
  unnest(cols = c(data)) %>%
  filter(Includes != "Eukaryotes + Prokaryotes") %>%
  unique() %>%
  ggplot() +
  geom_point(aes(x = log_Radius_um, y = log10(NitOx_count/GeneModels_count), color = Phylum)) + 
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = `flagcol_offset_model_regression_estimate_(Intercept)`/ln_log10_conversion, 
                  slope = `flagcol_offset_model_regression_estimate_.$log_Radius_um`/ln_log10_conversion, 
                  linetype = flagcol_offset_regression_type), 
              color = "black") +
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = (`flagcol_offset_model_regression_estimate_(Intercept)` + `flagcol_offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`flagcol_offset_model_regression_estimate_.$log_Radius_um` + `flagcol_offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "black") +
  geom_abline(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
              aes(intercept = (`flagcol_offset_model_regression_estimate_(Intercept)` - `flagcol_offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`flagcol_offset_model_regression_estimate_.$log_Radius_um` - `flagcol_offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "black") +
  geom_abline(aes(intercept = `offset_model_regression_estimate_(Intercept)`/ln_log10_conversion, 
                  slope = `offset_model_regression_estimate_.$log_Radius_um`/ln_log10_conversion, 
                  linetype = offset_regression_type), 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` + `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` + `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` - `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` - `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_text(data = . %>% filter(`offset_model_regression_p.value_.$log_Radius_um` < 0.05), 
            aes(x = 0.0, y = -5.1, label = paste("p-value =", signif(`flagcol_offset_model_regression_p.value_.$log_Radius_um`, digits = 2))), color = "black", check_overlap = TRUE) +
  geom_text(aes(x = 0.0, y = -4.9, label = paste("p-value =", signif(`offset_model_regression_p.value_.$log_Radius_um`, digits = 2))), color = "blue", check_overlap = TRUE) +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  scale_linetype_manual(name = "Regression type", values = c("poisson" = "solid" , "quasipoisson" = "twodash")) +
  scale_fill_manual(name = "Phylum", values =TaxaColors, guide = "none") +
  coord_cartesian(ylim = c(ROSMin, ROSMax)) +
  scale_shape_manual(values = c(21,19,23,18)) +
  facet_grid(cols = vars(NitOx), rows = vars(Includes)) +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text = element_text(size=11),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=11), 
        legend.title = element_text(size=13),
        strip.text = element_text(size=11)) +
  labs(x = ~log[10] ~ "(Median Cell Radius (µm))", 
       y = ~log[10] ~ "(Genes Encoding "~NO^`•` ~" Metabolizing Enzymes Normalized to Total Genes)")

ggsave(paste("Figure8",".png",sep = ""), plot = NitOxRadPlots, device = NULL, path = Figures,scale = 1, height=7, width= 10, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r NitOxRadPlots, fig.cap = NitOxRadPlots_cap, fig.height= 7, fig.width= 10, echo = FALSE}
NitOxRadPlots_cap <- glue("**Comparison of log~10~ (Total number of genes encoding ^•^NO metabolizing enzymes ('NitOx_count') normalized to the total number of genes present in each organism ('GeneModels_count')) vs. the log~10~ (median cell radius in µm ('log_Radius_um')).**", 
                            "Poisson (solid line) or Quasi-Poisson (dashed line) regressions fitted to data ± Standard Error (dotted line).",
                            "Regressions were run with (black line) or without (blue line) 'Colony' and 'Flagella' as co-variates.", 
                            "Selected prokaryote genomes are presented for comparison, but excluded from the presented regressions.",
                            "Symbol color corresponds to taxon lineage ('Phylum').", 
                            # "Citations for data sources can be found in Supplementary Table S3.", 
                          .sep = " ")

NitOxRadPlots
```

With increasing cell radius, eukaryotic phytoplankton do not have a varying fraction of their total genes to the capacity to produce ^•^NO, as seen in the `r NitOxRad$offset_regression_type[[1]]` regression in Figure \@ref(fig:NitOxRadPlots) (Blue line, Slope = `r ScientificTen(NitOxRad %>% filter(NitOx == "Production") %>% pluck('offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(NitOxRad %>% filter(NitOx == "Production") %>% pluck('offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p = `r ScientificTen(NitOxRad %>% filter(NitOx == "Production") %>% pluck('offset_model_regression', 1, "p.value", 2))`). Including prokaryotes in the regression in Figure \@ref(fig:NitOxRadPlots) does not substantially alter the interpretation.
We re-ran the `r NitOxRad$offset_regression_type[[1]]`, excluding those organisms that completely lack genes encoding enzymes for ^•^NO production (NitOx_count = 0, points along the x-axis), which resulted in a decreasing slope with increasing cell radius. Thus those phytoplankton with any detected capacity to produce ^•^NO indeed have a smaller fraction of their total genes encoding ^•^NO production with increasing radius (Figure \@ref(fig:NitOxRadPlots), Blue line, Slope = `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck('offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck('offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p value = `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck('offset_model_regression', 1, "p.value", 2))`). 
Including 'Flagella' and 'Colony' as co-factors to the regression that solely looks at organisms with the genomic capacity to produce ^•^NO resulted in a slope that is no longer significantly different from zero (Figure \@ref(fig:NitOxRadPlots), Black line, Slope = `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck('flagcol_offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck('flagcol_offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p = `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck('flagcol_offset_model_regression', 1, "p.value", 2))`), driven by the influence of 'Flagella' (p-value = `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck("flagcol_offset_model_pvalue", 1, tidy, "p.value", 3))`), but not 'Colony' (p-value = `r ScientificTen(NitOxRadNoZero %>% filter(NitOx == "Production") %>% pluck("flagcol_offset_model_pvalue", 1, tidy, "p.value", 4))`). 


With increasing cell radius, eukaryotic phytoplankton do not alter the fraction of their total genes to the capacity to scavenge ^•^NO, as seen in the `r NitOxRad$offset_regression_type[[2]]` regression slope not significantly different from zero (Figure \@ref(fig:NitOxRadPlots), Blue line, Slope = `r ScientificTen(NitOxRad %>% filter(NitOx == "Scavenging") %>% pluck('offset_model_regression', 1, "estimate", 2)/ln_log10_conversion)` ± `r ScientificTen(NitOxRad %>% filter(NitOx == "Scavenging") %>% pluck('offset_model_regression', 1, "std.error", 2)/ln_log10_conversion)`, p = `r ScientificTen(NitOxRad %>% filter(NitOx == "Scavenging") %>% pluck('offset_model_regression', 1, "p.value", 2))`). Including prokaryotes in the `r NitOxRadProk$offset_regression_type[[2]]` regression from Figure \@ref(fig:NitOxRadPlots) does not alter the interpretation, as the slope of the `r NitOxRad$offset_regression_type[[2]]` regression is still not significantly different from zero. We re-ran the `r NitOxRad$offset_regression_type[[2]]`, excluding those organisms that completely lack genes encoding enzymes for ^•^NO scavenging (NitOx_count = 0, points along the x-axis), did not contribute to explaining the fraction of total genes encoding ^•^NO scavenging with increasing radius. Non-enzymatic paths contribute to intracellular and extracellular ^•^NO production [@zweierNonenzymaticNitricOxide1999], and may explain the absences of genes encoding ^•^NO production across some genomes across all taxonomic lineages. Alternately, ^•^NO homeostastis may be achieved in some lineages by regulating the active cellular uptake and release of intracellular ^•^NO, as has been recently demonstrated in humans [@chenCellPenetratingDeliveryNitric2021]. Although nitric oxide dioxygenase sequences have only been identified once before from phytoplankton through metatranscriptomic analyses (in diatoms, haptophytes and dinoflagellates) [@lampeStrategiesPhytoplanktonResponse2019], there is limited understanding as to what may contribute to the active removal of ^•^NO, and the lack of ^•^NO scavenging genes across multiple organisms. More research is needed on NOD in the active removal of ^•^NO, as well as the NOS sequences detected in Synechococcus that also display nitric oxide dioxygenase-like activity [@piccianoNitricOxideSynthase2019]. Perhaps the low reactivity of ^•^NO does not warrant the active removal of ^•^NO as long as the concentration does not exceed the toxic threshold. This explanation is plausible given that *Platymonas helgolandica*, *Platymonas subcordiformis*, *Skeletonema costatum*, *Gymnodinium sp.*, and *Prorocentrum donghaiense* showed optimum growth in the presence of ^•^NO concentrations between 10^-9^ and 10^-6^ mol L^−1^ [@peifengEffectsNitricOxide2021], which are higher than the concentrations found in the ocean (Table \@ref(tab:ROSDiffusion)).

### Influences of Flagella, Colony Formation and Cell Shape on Gene Counts for Enzymes Metabolizing ^•^NO {.unnumbered}

```{r NitOxFlag_pval, include = FALSE}


NitOxFlagellaDelta <- MergedData %>%
  filter(NitOx == "Production" | NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(Flagella),
         !is.na(NitOx),
         Taxa != "Prokaryote") %>% 
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx) %>%
  unique() %>%
  pivot_wider(names_from = NitOx, values_from = NitOx_count) %>%
  mutate(prodNorm = Production/GeneModels_count,
         scavNorm = Scavenging/GeneModels_count) %>%
  group_by(Flagella) %>%
  summarize(Production = median(prodNorm), 
            Scavenging = median(scavNorm)) %>%
  mutate(deltaProduction = (Production[2] - Production[1])/ Production[1],
         deltaScavenging = (Scavenging[2] - Scavenging[1])/ Scavenging[1])

NitOxProdFlagPlot_pval <- MergedData %>%
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(NitOx == "Production",
         GeneModels_count != 0,
         !is.na(Flagella), 
         !is.na(NitOx),
         Taxa != "Prokaryote") %>%
  mutate(norm_NitOx = NitOx_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_NitOx ~ Flagella, data = ., method = "wilcox.test")

NitOxScavFlagPlot_pval <- MergedData %>%
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(Flagella), 
         !is.na(NitOx),
         Taxa != "Prokaryote") %>%
  mutate(norm_NitOx = NitOx_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_NitOx ~ Flagella, data = ., method = "wilcox.test")
```
 
```{r generate NitOxFlagPlot, include = FALSE}
NitOxFlagPlot <- MergedData %>%
  filter(NitOx == "Production" | NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(Flagella), 
         !is.na(NitOx),
         Taxa != "Prokaryote") %>% #Taxa != "Prokaryote"
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx) %>%
  unique() %>%
  ggplot(aes(x = Flagella, y = NitOx_count/GeneModels_count)) +
  geom_boxplot(notch = TRUE, outlier.shape = NA) +
  geom_jitter(aes(color = Phylum),width = 0.2, stat = "unique") +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  xlab("") +
  stat_compare_means(comparisons = MyFlagellaComparisons, method = "wilcox.test") +
  facet_grid(cols = vars(c(NitOx))) +
  scale_x_discrete(labels = c("no" = "No Flagella", "yes" = "Flagella")) +
  scale_y_continuous(labels = scales::scientific)  +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_text(size=9),
        axis.text.x = element_text(size=12),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=10), 
        legend.title = element_text(size=10),
        strip.text = element_text(size=11)) +
  labs(y = "Genes Encoding " ~NO^`•`~" Metabolizing Enzymes Normalized to Total Genes")

# ggsave(paste("HyPePennateCentricPlot",".png",sep = ""), plot = HyPePennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

# NitOxFlagPlot
```

```{r NitOxFlagPlot, fig.cap = NitOxFlagPlot_cap, fig.height= 5, fig.width= 5, echo = FALSE}
NitOxFlagPlot_cap <- glue("**Comparison of total number of genes encoding ^•^NO metabolizing enzymes ('NitOx_count') normalized to the total number of genes present in each organism ('GeneModels_count') vs. the presence or absence of flagella in the organism.**", 
                                       "Symbol color corresponds to taxon lineage ('Phylum').", 
                          "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                                       "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
# NitOxFlagPlot
```

Setting aside the influence of cell size, non-flagellated and flagellated phytoplankton do not show statistically significant differences in the fractions of total gene content encoding ^•^NO production (p = `r ScientificTen(NitOxProdFlagPlot_pval$p)`) nor  ^•^NO scavenging (p = `r ScientificTen(NitOxScavFlagPlot_pval$p)`) (Data not presented), suggesting ^•^NO does not have a generalized interaction with flagella across eukaryotic phytoplankton groups.

```{r NitOxColony_pval, include = FALSE}

NitOxColonyDelta <- MergedData %>%
  filter(NitOx == "Production" | NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(ColonySpecies),
         Taxa != "Prokaryote") %>% 
  select(FileName, NitOx, NitOx_count, GeneModels_count, ColonySpecies) %>%
  unique() %>%
  pivot_wider(names_from = NitOx, values_from = NitOx_count) %>%
  mutate(prodNorm = Production/GeneModels_count,
         scavNorm = Scavenging/GeneModels_count) %>%
  group_by(ColonySpecies) %>%
  summarize(Production = median(prodNorm), 
            Scavenging = median(scavNorm)) %>%
  mutate(deltaProduction = (Production[2] - Production[1])/ Production[1],
         deltaScavenging = (Scavenging[2] - Scavenging[1])/ Scavenging[1])

NitOxProdColonyPlot_pval <- MergedData %>%
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(NitOx == "Production",
         GeneModels_count != 0,
         !is.na(ColonySpecies),
         Taxa != "Prokaryote") %>%
  mutate(norm_NitOx = NitOx_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_NitOx ~ ColonySpecies, data = ., method = "wilcox.test")

NitOxScavColonyPlot_pval <- MergedData %>%
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(ColonySpecies),
         Taxa != "Prokaryote") %>%
  mutate(norm_NitOx = NitOx_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_NitOx ~ ColonySpecies, data = ., method = "wilcox.test")
```

```{r generate NitOxColonyPlot, include = FALSE}
NitOxColonyPlot <- MergedData %>%
  filter(NitOx == "Production" | NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(ColonySpecies),
         Taxa != "Prokaryote") %>%
  select(FileName, ColonySpecies, NitOx_count, GeneModels_count, Phylum, NitOx) %>%
  unique() %>%
  ggplot(aes(x = ColonySpecies, y = NitOx_count/GeneModels_count)) +
  geom_boxplot(notch = TRUE, outlier.shape = NA) +
  geom_jitter(aes(color = Phylum),width = 0.2, stat = "unique") +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  xlab("") +
  stat_compare_means(comparisons = MyColonyComparisons, method = "wilcox.test") +
  facet_grid(cols = vars(c(NitOx))) +
  scale_x_discrete(labels = c("FALSE" = "Non-colony", "TRUE" = "Colony")) +
  scale_y_continuous(labels = scales::scientific)  +
  scale_colour_manual(name = "Phylum", values =TaxaColors) +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_text(size=9),
        axis.text.x = element_text(size=12),
        axis.title = element_text(size=12),
        legend.text = element_text(size=10),
        legend.title = element_text(size=10),
        strip.text = element_text(size=11)) +
  labs(y = "Genes Encoding " ~NO^`•`~" Metabolizing Enzymes Normalized to Total Genes")

# ggsave(paste("HyPePennateCentricPlot",".png",sep = ""), plot = HyPePennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

# NitOxColonyPlot
```

```{r NitOxColonyPlot, fig.cap = NitOxColonyPlot_cap, fig.height= 5, fig.width= 5, echo = FALSE}
NitOxColonyPlot_cap <- glue("**Comparison of total number of genes encoding ^•^NO metabolizing enzymes ('NitOx_count') normalized to the total number of genes present in each organism ('GeneModels_count') vs. the ability of the organism to form colonies.**", 
                                       "Symbol color corresponds to taxon lineage ('Phylum').", 
                            "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                                       "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
# NitOxColonyPlot
```


Comparing non-colony to colony forming phytoplankton does not show a statistically significant difference in the fraction of total gene content encoding ^•^NO production (p = `r ScientificTen(NitOxProdColonyPlot_pval$p)`) nor  ^•^NO scavenging (p = `r ScientificTen(NitOxScavColonyPlot_pval$p)`) (Data not presented), suggesting ^•^NO metabolism does not have a generalized role in colony formation across eukaryotic phytoplankton groups.


```{r NitOxDiatomShape_pval, include = FALSE}
NitOxPennateCentricDelta <- MergedData %>%
  filter(Taxa == "Diatom",
         NitOx == "Production" | NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(PennateCentric),
         Taxa != "Prokaryote") %>% 
  select(FileName, NitOx, NitOx_count, GeneModels_count, PennateCentric) %>%
  unique() %>%
  pivot_wider(names_from = NitOx, values_from = NitOx_count) %>%
  mutate(prodNorm = Production/GeneModels_count,
         scavNorm = Scavenging/GeneModels_count) %>%
  group_by(PennateCentric) %>%
  summarize(Production = median(prodNorm), 
            Scavenging = median(scavNorm)) %>%
  mutate(deltaProduction = (Production[2] - Production[1])/ Production[1],
         deltaScavenging = (Scavenging[2] - Scavenging[1])/ Scavenging[1])

NitOxProdPennateCentricPlot_pval <- MergedData %>%
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(Taxa == "Diatom",
         NitOx == "Production",
         GeneModels_count != 0,
         !is.na(PennateCentric)) %>%
  mutate(norm_NitOx = NitOx_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_NitOx ~ PennateCentric, data = ., method = "wilcox.test")

NitOxScavPennateCentricPlot_pval <- MergedData %>%
  select(FileName, Flagella, NitOx_count, GeneModels_count, Phylum, NitOx, Genus, Taxa, ColonySpecies, PennateCentric) %>%
  filter(Taxa == "Diatom",
         NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(PennateCentric)) %>%
  mutate(norm_NitOx = NitOx_count/GeneModels_count) %>%
  unique() %>%
  compare_means(norm_NitOx ~ PennateCentric, data = ., method = "wilcox.test")
```

```{r generate NitOxPennateCentricPlot, include = FALSE}
NitOxPennateCentricPlot <- MergedData %>%
  filter(Taxa == "Diatom",
         NitOx == "Production" | NitOx == "Scavenging",
         GeneModels_count != 0,
         !is.na(PennateCentric),) %>%
  select(FileName, PennateCentric, NitOx_count, GeneModels_count, Phylum, NitOx, Genus) %>%
  unique() %>%
  ggplot(aes(x = PennateCentric, y = NitOx_count/GeneModels_count)) +
  geom_boxplot(notch = TRUE, outlier.shape = NA) +
  geom_jitter(aes(colour = Genus),width = 0.2, stat = "unique") +
  # coord_cartesian(ylim = c(ROSBoxPlotMin, ROSBoxPlotMax)) +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  xlab("") +
  facet_grid(cols = vars(NitOx)) +
  stat_compare_means(comparisons = MyShapeComparisons, method = "wilcox.test") +
  # ggforce::facet_zoom(ylim = c(0, ROSBoxPlotMax)) +
  scale_y_continuous(labels = scales::scientific) +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text.y = element_text(size=9),
        axis.text.x = element_text(size=12),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=10), 
        legend.title = element_text(size=10),
        strip.text = element_text(size=11)) +
  labs(y = "Genes Encoding" ~NO^`•` ~" Metabolizing Enzymes Normalized to Total Genes")

# ggsave(paste("NitOxPennateCentricPlot",".png",sep = ""), plot = NitOxPennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

ggsave(paste("Figure9",".png",sep = ""), plot = NitOxPennateCentricPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

```

```{r NitOxPennateCentricPlot, fig.cap = NitOxPennateCentricPlot_cap, fig.height= 7, fig.width= 7, echo = FALSE}
NitOxPennateCentricPlot_cap <- glue("**Comparison of total number of genes encoding ^•^NO metabolizing enzymes ('NitOx_count') normalized to the total number of genes present in each diatom ('GeneModels_count') vs. the growth form of the diatom ('PennateCentric').**", 
                                       "Symbol color corresponds to taxon lineage ('Phylum').", 
                                    "Notch spans ± standard error of the median.",
                           "Box spans median ± 1 quartile of the data.",
                           "Whiskers span the range excluding outliers in the data.",
                                       "Citations for data sources can be found in Supplementary Table S3.", .sep = " ")
NitOxPennateCentricPlot
```

Most centric diatoms carry genes annotated as encoding ^•^NO producing enzymes, whereas most pennate diatoms do not (p = `r ScientificTen(NitOxProdPennateCentricPlot_pval$p)`). In contrast, most centric diatoms lack genes annotated as encoding ^•^NO scavenging enzymes, whereas most centric diatoms carry those genes (p = `r ScientificTen(NitOxScavPennateCentricPlot_pval$p)`) (Figure \@ref(fig:NitOxPennateCentricPlot)). 

The larger fractional gene allocation to ^•^NO production, and smaller fraction of genes that encode  ^•^NO scavenging enzymes, in centric diatoms (Figure \@ref(fig:NitOxPennateCentricPlot)) counters our hypothesis that diffusion from pennate diatoms would drive gene allocations in favor of ^•^NO production (Hypothesis \@ref(hyp:HyPeNitOxDiatom)). Given the strong contrast in annotated ^•^NO metabolism genes, it is likely that ^•^NO has regulatory or signaling roles that vary systematically between pennate and centric diatoms, outside any diffusional influences. For example, ^•^NO inhibits diatom adhesion to substrate [@thompsonRoleNitricOxide2008; @vardiDiatomGeneRegulating2008]. Pennates are more likely to grow adhered in biofilms [@hunsuckerSituMeasurementsDiatom2016], which may explain the striking differences in total gene allocation to  ^•^NO production and scavenging. Alternately, @didatoTranscriptomeSequencingThree2015 identified putative nitric oxide synthase sequences in the transcriptomes of three Pennate Diatom species (*Pseudo-nitzschia arenysensis*, *Pseudo-nitzschia delicatissima* and *Pseudo-nitzschia multistriata*), so it is possible the apparent lack of ^•^NO producing sequences in pennates is due to errors in the unsupervised annotations from eggNOG.

XXX perhaps reiterate the importance of NO in cell signaling. Cell signaling likely explains the differences in cell shape between pennate and centric diatoms. To respond to reviewer #1's comments "Do the results in Figure 9（Lines 591-614） refute Hypothesis 2（Lines 186-196）, Hypothesis 3（Lines 198-211）, and Hypothesis 4（Lines 213-223） simultaneously，and explain the relationship between them."XXX

## Summary {.unnumbered}

H~2~O~2~, O~2~^•−^ and ^•^NO have multiple intracellular and extracellular roles across phytoplankton taxa [@roeUptakeMechanismsInorganic2014; @delrioReactiveOxygenSpecies2009; @sutherlandExtracellularSuperoxideProduction2019; @delrioReactiveOxygenSpecies2009; @delrioROSGenerationPeroxisomes2016]. The differential reactivities, diffusion distances, diffusibilities across cell membranes, and roles in cell signaling of H~2~O~2~, O~2~^•−^ and ^•^NO (Table \@ref(tab:ROSDiffusion)) influence genomic allocation patterns for the production and scavenging of these three ROS.

Our work analyzed the fraction of the total genes in a genome associated with the metabolism of a particular ROS. The presence or absence of genes encoding specific ROS metabolizing enzymes may be caused by genetic drift, or may relate to a selective advantage linked to other metabolites of the same enzyme, rather than an enzymatic role in ROS metabolism, *per se*.  Furthermore, the presence of a gene in a genome does not necessarily mean the encoded enzyme will be active, and closely related enzymes may mediate different activities in different organisms. The influence of non-enzymatic pathways such as carotenoids or tocopherols may affect the hypotheses listed above, but were beyond the frame of this study.

Patterns of genomic allocations to production and scavenging of different ROS vary significantly by substrate.

O~2~^•−^ has relatively high reactivity, short intracellular and extracellular lifetimes and limited cell membrane crossing. We did not find genes specifically encoding O~2~^•−^ production in eukaryotic phytoplankton genomes. As expected, genes encoding O~2~^•−^ scavenging were ubiquitous, but the fractional gene allocation to O~2~^•−^ scavenging decreases as cell radius (and the co-varying total gene count) increases, consistent with a nearly fixed set of core genes that do not change with increasing gene count. 

H~2~O~2~ has low reactivity, long intracellular and extracellular lifetimes and readily crosses cell membranes.  Across eukaryotic phytoplankton, fraction of the total genes encoding H~2~O~2~ producing enzymes decreases with increasing cell radius, consistent with maintenance of ROS homeostasis in the face of slower diffusional losses from larger cells. The fraction of the total genes encoding H~2~O~2~ scavenging enzymes also decreases with increasing cell radius, but with a slope smaller than that for H~2~O~2~ producing enzymes. Presence of flagella and colony formation appear to influence H~2~O~2~ metabolism.

^•^NO has low reactivity, long intracellular and extracellular lifetimes and readily crosses cell membranes. Neither the fraction of the total genes for ^•^NO production nor for scavenging changed significantly with increasing cell radius, consistent with relatively low cytotoxicity and roles of ^•^NO in taxonomically diverse regulatory systems. Pennate diatoms frequently lack genes annotated as encoding ^•^NO producing enzymes, whereas centric diatoms frequently lack genes annotated as encoding ^•^NO scavenging enzymes. This finding is not explicable by differential diffusional losses of ^•^NO, but may reflect distinct roles of ^•^NO in the regulatory systems of diatom lineages.


# Open Data Statement {.unnumbered}

Code, genes annotated for ROS metabolism (MergedData.csv), MetaData.csv and DataDictionary.csv are stored on https://github.com/NaamanOmar/ROS_bioinfo/tree/master/ROSGenomicPatternsAcrossMarinePhytoplankton.

Annotations of all genes from genomes or transcriptomes of organisms used in this study, MetaData.csv and DataDictionary.csv are stored on DRYAD (DOI pending) to facilitate reuse for other purposes.

# Supplementary Materials {.unnumbered}

## Supplementary Figures {.unnumbered}

```{r O to N/A for dotplot, include = FALSE}
#create a new data frame for the dot plot, this one will take all 0's and make them N/As so that they are not plotted (in other plots, 0 is still a data point)
MergedData_na <- MergedData 
MergedData_na[MergedData==0]=NA
ROSEnzymeLevels <- WideData %>% 
  select(-c(Genus, species, Strain, Taxa, Ome)) %>% 
  colSums() %>% 
  enframe() %>% 
  arrange(desc(value))
MergedData_na <- MergedData_na %>%
  left_join(., ROSEnzymeLevels, by = c("EnzymeName" = "name")) %>%
  rename(ROSEnzyme_TotalCount = value) %>%
  mutate(EnzymeName = factor(.$EnzymeName, levels = ROSEnzymeLevels$name))
# MergedData_na$Enzyme_parent_ASA <-  factor(MergedData_na$EnzymeParent, levels= c("Superoxide_dismutase","Glutathione_reductase","Ascorbate_cycle" ,"Peroxidase","Peroxiredoxin","Catalase", "NO_sythesis", "Oxidase"))
MergedData_na$taxon_facet <- factor(MergedData_na$Taxa, levels= c("Cryptophyte","Diatom","Dinoflagellate" ,"Eustigmatophyte","Green","Haptophyte", "Pelagophyte", "Phaeista", "Red", "Prokaryote"))
```

```{r RadTaxaviolin, fig.cap=RadTaxaviolin_cap, echo = FALSE}
RadTaxaviolin_cap <- glue("**Violin plot presenting the range of log~10~ of the median cell radius in µm ('log_Radius_um') for each taxonomic lineage ('Phylum').**", 
                              "Point colour corresponds to the source of the data, whether Genome or Transcriptome ('Ome').", 
                              "Violin width indicates the fraction of all datapoints occuring at a cell radius ('log_Radius_um') within a phylum.",
                              "Citations for data sources are in Supplementary Table S3.",
                              .sep = " ")

RadTaxaviolin <- MergedData %>% 
  # filter(Taxa != "Prokaryote") %>%
  ggplot(aes(x= reorder(Phylum, log_Radius_um), y = log_Radius_um))+
  geom_violin(scale = "area")+
  geom_jitter(aes(shape = Ome, color = Phylum), stat = "unique", width = 0.2) +
  theme_bw() +  
  scale_shape_manual(values = c(19,1)) +
  scale_color_manual(values = TaxaColors) +
  # scale_y_continuous(limits = c(-0.6, 1.7)) %>%
  # theme(legend.position = "bottom") +
  coord_flip() +
  guides(color="none") +
  labs(x = "Phylum",
       y = ~log[10] ~ "(Median Cell Radius (µm))")

RadTaxaviolin

ggsave(paste("FigureS1",".png",sep = ""), plot = RadTaxaviolin, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r generate LongLatMap, include = FALSE}
LongLatMap_cap <- glue("Longitude and Latitude of isolation of analyzed organisms, overlaid on a world map",
                       "Point colour corresponds to the taxonomic lineage ('Phylum').",
                       "Ocean colour corresponds to depth",
                       "Citations for data sources are in Supplementary Table S3.",
                       "Data used to generate world map produced from the 'ggOceanMaps' R package [@vihtakariGgOceanMapsPlotData2021]", .sep = " ")

LongLatMap <- ggOceanMaps::basemap(data = data.frame(lon = c(-180, 0, 180), lat = c(90, 0, -90)), bathymetry = TRUE, lon.interval = 45) +
  geom_point(data = MergedData, aes(x = Longitude, y = Latitude, color = Phylum)) +
  coord_sf(expand = F)+
  scale_colour_manual(values =TaxaColors) +
  theme_bw() 

ggsave(paste("FigureS2",".png",sep = ""), plot = LongLatMap, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r LongLatMap, fig.cap = LongLatMap_cap, fig.height = 7, fig.width = 7, echo = FALSE}
LongLatMap
```

```{r ManualAutoCompare, fig.cap = ManualAutoCompare_cap, fig.align = 'centre',out.width='100%', out.height='100%', echo = FALSE}
ManualAutoCompare_cap <- glue::glue("**Comparison of paired counts of particular genes encoding ROS Production or Scavenging from manual and automatic annotations taken from the same organism.**", 
                                    "Data was drawn from a subset of genomes and transcriptomes which were both manually and automatically annotated.",
                                    "Colour corresponds to the 'Gene'",
                                    "Points are jittered to avoid overlapping, resulting in blocks around frequently occuring counts.",
                                    "Dashed line is placed at 1:1 where Manual and Automated counts would be equal.", 
                                    "Citations for data sources in Supplemental Data Table S4.", .sep = " ")

ManualAutoCompare <- ManualAutoCompareData %>%
  select(-Automated_count) %>%
  rename(Automated_count = AutomatedECKO_count) %>%
  ggplot() +
  geom_jitter(aes(x = Manual_count, y = Automated_count, color = ROSEnzymeAbbrev)) +
  geom_abline(aes(intercept = 0, slope = 1), linetype = "dashed") +
  coord_fixed(ratio = 1) +
  labs(x = "Manual Gene Count", 
       y = "Automatic Gene Count",
       color = "Gene") +
  theme_bw()

ManualAutoCompare

ggsave(paste("FigureS3",".png",sep = ""), plot = ManualAutoCompare, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```



```{r generate PoissonDist, include = FALSE}
PoissonDist_cap <- glue("**Histogram of occurrences of number of total genes, in a genome or transcriptome, (y axis) that code for the production of enzymes that produce or scavenge H~2~O~2~, O~2~^•−^ or ^•^NO *in vivo*.**",
                        "Symbol color corresponds to taxon lineage ('Taxa')", .sep = " ")
#                            "Symbol color corresponds to taxon lineage ('Taxa')", .sep = " "))

PoissonDist <- MergedData %>% 
  select(FileName, HyPe, HyPe_count, SupOx, SupOx_count, NitOx, NitOx_count, Phylum) %>%
  unique() %>%
  pivot_longer(c(HyPe, SupOx, NitOx), names_to = "ROS", values_to = "ProdScav") %>% 
  pivot_longer(c(HyPe_count, SupOx_count, NitOx_count), names_to = "ROS_countType", values_to = "ROS_count") %>%
  select(FileName, ProdScav, ROS_countType, ROS_count, Phylum, ROS) %>%
  filter(!is.na(ProdScav)) %>%
  unique() %>%
  ggplot() + 
  geom_histogram(aes(x= ROS_count, fill = Phylum), binwidth = 4) +
  facet_grid(rows = vars(ROS), cols = vars(ProdScav), scales = "free_y") +
  scale_fill_manual(values =TaxaColors) +
  theme_bw() +
  labs(x = "Number of Genes Encoding Metabolism of a Particular ROS",
       y = "Frequency")

ggsave(paste("FigureS4",".png",sep = ""), plot = PoissonDist, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r PoissonDist, fig.cap = PoissonDist_cap, echo = FALSE}
PoissonDist
```

```{r, include = FALSE}
OmesJust <- OmesPairs %>%
  select(EnzymeName, ECNumber, ROSGene_count.g, ROSGene_count.t, FileName.g) %>%
  mutate(EnzymeNameFileName = interaction(EnzymeName, FileName.g)) 
  # pivot_longer(c(ROSGene_count.g, ROSGene_count.t), names_to = "source", values_to = "ROSGene_count") #%>%
  # head(100) #%>%
  # compare_means(ROSGene_count ~ EnzymeNameFileName, paired = TRUE, data = ., method = "wilcox.test")


# OmesPairs %>%
#   select(EnzymeName, ECNumber, ROSGene_count.g, ROSGene_count.t, FileName.g) %>%
#   mutate(EnzymeNameFileName = interaction(EnzymeName, FileName.g)) %>%
#   pivot_longer(c(ROSGene_count.g, ROSGene_count.t), names_to = "source", values_to = "ROSGene_count") %>%
#   head(100) %>%
#   ggplot() +
#   geom_point(aes(x = source, y = ROSGene_count, ), show.legend = "none") +
#   geom_count(aes(x = source, y = ROSGene_count), show.legend = "none") 
```


```{r generate OmesPlot, include = FALSE}
OmesPlot <- OmesPairs %>%
  ggplot() + 
  geom_jitter(aes(x = ROSGene_count.g, y = ROSGene_count.t, colour = Phylum, ),)  +#size = as.numeric(GenomeSize_mbp.g)  stat = "unique"
  # geom_count(aes(x = ROSGene_count.g, y = ROSGene_count.t, colour = Phylum, ),)  +#size = as.numeric(GenomeSize_mbp.g)  stat = "unique"
  # geom_point(aes(x = ROSGene_count.g, y = ROSGene_count.t, size = Omes_pairs_comparison)) +
  geom_abline(intercept = 0, slope =  1, linetype = "dashed") +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text=element_text(size=12),
        axis.title=element_text(size=16), 
        aspect.ratio = 1,
        legend.text=element_text(size=9), 
        legend.title = element_text(size=10)) +
  guides(colour=guide_legend(title = "Phylum", override.aes = list(size=4)), size = guide_legend(title = "Genome Size (MBp)")) +
  scale_colour_manual(values =TaxaColors) +
  # scale_radius(breaks=c(2,4,6,8,10,12,15,20,25)) +
  # xlab("ROS gene count by family (genome)") +
  # ylab("ROS gene count by family (transcriptome)") +
  coord_fixed(ratio = 1) +
  scale_x_continuous(limits = c(0,28)) +
  scale_y_continuous(limits = c(0,28)) +
  labs(x = "Counts of Genes from Genome", 
       y = "Counts of Genes from Transcriptome")
  # ggforce::facet_zoom(xlim = c(0,10), ylim = c(0,10), zoom.size = 0.25)

OmesPlot_cap <- glue("**Comparison of paired counts of particular genes encoding ROS Production or Scavenging from the genome ('ROSGene_count.g') or transcriptome ('ROSGene_count.t') taken from the same organism.**",
                        "Data was drawn from a subset of analyzed organisms for which both genome and transcriptome were available.",
                       "Colour corresponds to the taxonomic lineage ('Phylum')",
                       "Points are jittered to avoid overlapping, resulting in blocks around frequently occuring counts.",
                       "Dashed line is at 1:1 where 'ROSGene_count.g' and 'ROSGene_count.t' would be equal.", 
                     "Citations for data sources are in Supplementary Table S3.",
                    .sep = " ")

ggsave(paste("FigureS5",".png",sep = ""), plot = OmesPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)

```

```{r OmesPlot, fig.cap = OmesPlot_cap, fig.height= 7, fig.width= 7, echo=FALSE}
OmesPlot

# ggsave(paste("OmesPlot",".png",sep = ""), plot = OmesPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r GeneModelsLatPlot, fig.cap = GeneModelsLatPlot_cap, echo=FALSE}
# GeneModelsLatPlot <- MergedData %>%
#   # filter(Taxa != "Prokaryote") %>%
#   filter(GeneModels_count != "0") %>%
#   ggplot(aes(x=abs_Latitude, y= log_GeneModels_count)) +
#   geom_point(aes(shape = Ome, colour = Taxa), size = 3)+
#   ggalt::geom_encircle(aes(color = Phylum), expand = 0.01) +
#   scale_colour_manual(values =TaxaColors)+
#   theme_bw()+
#   theme(panel.grid.major = element_blank(),
#         panel.grid.minor = element_blank(),
#         axis.text=element_text(size=12),
#         axis.title=element_text(size=16),
#         aspect.ratio = 1,
#         legend.text=element_text(size=10),
#         legend.title = element_text(size=12))#+
#   # xlab("log Radius, µm")+
#   # ylab("log Genome Size, mbp")
# 
# GeneModelsLatPlot

GeneModelsLatPlot_cap <- glue("**Comparison of log10 of the total number of genes in an organism ('log_GeneModels_count') with the absolute value of the strains Latitude of isolation ('abs_Latitude').**", 
                              "Colour corresponds to the taxonomic lineage ('Taxa'), whereas symbol shape corresponds to the source of the data, whether Genome or Transcriptome ('Ome').", 
                              "Citations for data sources are in Supplementary Table S3.",
                              .sep = " ")
```

```{r HyPeShapiro, include = FALSE}
HyPeProdShapiro <- MergedData %>% 
  select(FileName, HyPe,HyPe_count) %>%
  unique() %>%
  filter(HyPe == "Production") %>%
  .$HyPe_count %>%
  shapiro.test() %>%
  tidy()

HyPeScavShapiro <- MergedData %>% 
  select(FileName, HyPe,HyPe_count) %>%
  unique() %>%
  filter(HyPe == "Scavenging") %>%
  .$HyPe_count %>%
  shapiro.test() %>% 
  tidy()
```

```{r SupOxScavShapiro, include = FALSE}
SupOxScavShapiro <- MergedData %>% 
  filter(SupOx == "Scavenging") %>%
  select(FileName, SupOx, SupOx_count) %>%
  unique() %>%
  # filter(SupOx == "Production") %>%
  .$SupOx_count %>%
  shapiro.test() %>%
  tidy()
```


```{r NitOxShapiro, include = FALSE}
NitOxProdShapiro <- MergedData %>% 
  filter(NitOx == "Production") %>%
  select(FileName, NitOx, NitOx_count) %>%
  unique() %>%
  # filter(NitOx == "Production") %>%
  .$NitOx_count %>%
  shapiro.test() %>%
  tidy()

NitOxScavShapiro <- MergedData %>% 
  filter(NitOx == "Scavenging") %>%
  select(FileName, NitOx, NitOx_count) %>%
  unique() %>%
  # filter(NitOx == "Production") %>%
  .$NitOx_count %>%
  shapiro.test() %>%
  tidy()
```


```{r generate OchrophyteRadPlot, include=FALSE}
OchrophyteHyPeRad$Includes <- c("Ochrophyte Regressions")
OchrophyteNitOxRad$Includes <- c("Ochrophyte Regressions")
OchrophyteSupOxRad$Includes <- c("Ochrophyte Regressions")

OchrophyteSupOxRad <- OchrophyteSupOxRad %>% 
  mutate(ROSType = "SupOx",
         ROSName = "O~2~^•−^") %>%
  rename(ROSMetabolism = SupOx)

OchrophyteNitOxRad <- OchrophyteNitOxRad %>% 
  mutate(ROSType = "NitOx",
         ROSName = "^•^NO") %>%
  rename(ROSMetabolism = NitOx)

OchrophyteRadPlot <-  rbind(OchrophyteSupOxRad, OchrophyteHyPeRad, OchrophyteNitOxRad) %>%
  # filter(HyPe == "Production" | HyPe == "Scavenging") %>%
  unnest(c(offset_model_regression), names_sep = "_") %>%
  unnest(c(flagcol_offset_model_regression), names_sep = "_") %>%
  pivot_wider(., names_from = c(flagcol_offset_model_regression_term), values_from = c(flagcol_offset_model_regression_estimate, flagcol_offset_model_regression_std.error, flagcol_offset_model_regression_statistic, flagcol_offset_model_regression_p.value)) %>%
  pivot_wider(., names_from = c(offset_model_regression_term), values_from = c(offset_model_regression_estimate, 
                                                                               offset_model_regression_std.error, offset_model_regression_statistic, offset_model_regression_p.value)) %>%
  unnest(c(offset_model_pvalue), names_sep = "_") %>%
  unnest(c(flagcol_offset_model_pvalue), names_sep = "_") %>%
  unnest(cols = c(data)) %>%
  ggplot() +
  geom_point(aes(x = log_Radius_um, y = log10(HyPe_count/GeneModels_count), color = Phylum)) +
  geom_point(aes(x = log_Radius_um, y = log10(NitOx_count/GeneModels_count), color = Phylum)) +
  geom_point(aes(x = log_Radius_um, y = log10(SupOx_count/GeneModels_count), color = Phylum)) +
  geom_abline(aes(intercept = `offset_model_regression_estimate_(Intercept)`/ln_log10_conversion, 
                  slope = `offset_model_regression_estimate_.$log_Radius_um`/ln_log10_conversion, 
                  linetype = offset_regression_type), 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` + `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` + `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_abline(aes(intercept = (`offset_model_regression_estimate_(Intercept)` - `offset_model_regression_std.error_(Intercept)`) /ln_log10_conversion, 
                  slope = (`offset_model_regression_estimate_.$log_Radius_um` - `offset_model_regression_std.error_.$log_Radius_um`)/ln_log10_conversion), 
              linetype = "dotted", 
              color = "blue") +
  geom_text(aes(x = 1.1, y = -4.1, label = paste("p-value =", signif(`offset_model_regression_p.value_.$log_Radius_um`, digits = 2))), color = "blue", check_overlap = TRUE) +
  facet_grid(cols = vars(ROSMetabolism), rows = vars(ROSType), ) +
  scale_colour_manual(name = "Phylum", values =TaxaColors) + 
  
  scale_linetype_manual(name = "Regression type", values = c("poisson" = "solid" , "quasipoisson" = "twodash")) +
  scale_shape_manual(values = c(21,19,23,18)) +
  coord_cartesian(ylim = c(ROSMin, ROSMax)) +
  guides(color = "none") +
  # scale_x_continuous(limits = c(-0.5, 1.6)) +
  theme_bw() +
  theme(panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        axis.text = element_text(size=11),
        axis.title = element_text(size=12), 
        legend.text = element_text(size=11), 
        legend.title = element_text(size=13),
        strip.text = element_text(size=11)) +
  labs(x = ~log[10] ~ "(Median Cell Radius (µm))", 
       y = ~log[10] ~ "(Genes Encoding ROS Metabolizing Enzymes Normalized to Total Genes)") 

ggsave(paste("FigureS6",".png",sep = ""), plot = OchrophyteRadPlot, device = NULL, path = Figures,scale = 1, height=7, width= 7, units = c("in"),dpi = 300, limitsize = TRUE)
```

```{r OchrophyteRadPlot, fig.cap = OchrophyteRadPlot_cap, fig.height= 6, fig.width= 10, echo = FALSE}
OchrophyteRadPlot_cap <- glue("**Comparison of log~10~ (Total number of genes encoding H~2~O~2~, O~2~^•−^ or ^•^NO metabolizing enzymes normalized to the total number of genes present in each Ochrophyte) vs. the log~10~( median cell radius in µm).**",
                            "Poisson (solid line) or Quasi-Poisson (dashed line) regressions fitted to data ± Standard Error (dotted line).",
                            "Regressions were run without (blue line) 'Colony' and 'Flagella' as co-variates.", 
                            "Citations for data sources are in Supplementary Table S3.",
                         .sep = " ")

OchrophyteRadPlot
```

```{r EukHyPeDotPlot, fig.height = 20, fig.width = 10}
EukHyPeDotPlot <- MergedData_na %>%
  filter(HyPe == "Production" | HyPe == "Scavenging", 
         Taxa != "Prokaryote",
         ROSEnzyme_TotalCount != 0,
         GeneModels_count != 0) %>%
  ggplot() +
  geom_point(aes(x = EnzymeName, y= reorder(Name, -Radius_um), size = ROSGene_count, color = Taxa, shape = Ome)) +
  scale_x_discrete(position ="top") +
  scale_color_manual(values = TaxaColors) +
  scale_shape_manual(values = c(16,1))+
  scale_radius(breaks=c(2,4,6,8,10,12,15,20,25)) +
  ggtitle("HyPe") +
  theme_bw() +
  # theme(axis.text.x = element_text(angle = 90, size = 10),
  #       axis.text.y = element_text(face="italic",size = 10, vjust = 0)) +
  facet_grid(rows = vars(Taxa), cols = vars(HyPe), scales = "free", space = "free") +
  ggpubr::rotate_x_text(hjust = 0)# +
  # theme(axis.text.x = element_text(face = ifelse(levels(MergedData_na$EnzymeName %in% MergedData_na$HyPe_PriSec), "bold", "italic")))
  # theme(axis.text.x = ifelse(MergedData_na$HyPe_PriSec == "Primary", element_text(face = "bold"), NULL))
  # ifelse(MergedData_na$HyPe_PriSec == "Primary", theme(axis.text.x = element_text(face = "bold")), NULL)
  # ifelse(MergedData_na$HyPe_PriSec == "Primary" ~ theme(axis.text.x = element_text(face = "bold")),
            # NULL)
# theme(axis.text.x = element_text(face = "bold"))
    # axis.text.x = element_text(face = ifelse(ROSEnzymes$HyPe_Prisec == "Primary", "bold", NULL)))
EukHyPeDotPlot
# saveRDS(EukHyPeDotPlot, file = file.path(Figures, "EukHyPeDotPlot.Rds"))
```

```{r ProkHyPeDotPlot, fig.height = 10, fig.width = 10}
ProkHyPeDotPlot <- MergedData_na %>%
  filter(HyPe == "Production" | HyPe == "Scavenging", 
         Taxa == "Prokaryote",
         ROSEnzyme_TotalCount != 0, 
         GeneModels_count != 0) %>%
  ggplot() +
  geom_point(aes(x = EnzymeName, y= reorder(interaction(Genus, species, Strain), -Radius_um), size = ROSGene_count, color = Genus, shape = Ome)) +
  scale_x_discrete(position ="top") +
  # scale_color_manual(values = TaxaColors) +
  scale_shape_manual(values = c(16,1))+
  scale_radius(breaks=c(1,2,3,4,5)) +
  ggtitle("HyPe") +
  theme_bw() +
  # theme(axis.text.x = element_text(angle = 90, size = 10),
  #       axis.text.y = element_text(face="italic",size = 10, vjust = 0)) +
  facet_grid(rows = vars(Taxa), cols = vars(HyPe), scales = "free", space = "free") +
  ggpubr::rotate_x_text(hjust = 0) #+
  # theme(panel.grid.major = element_blank(),
  #       panel.grid.minor = element_blank())
ProkHyPeDotPlot
# saveRDS(ProkHyPeDotPlot, file = file.path(Figures, "ProkHyPeDotPlot.Rds"))
```

```{r EukSupOxDotPlot, fig.height = 20, fig.width = 10}
EukSupOxDotPlot <- MergedData_na %>%
  filter(SupOx == "Scavenging" | SupOx == "Production", 
         Taxa != "Prokaryote",
         ROSEnzyme_TotalCount != 0, 
         GeneModels_count != 0) %>%
  ggplot() +
  geom_point(aes(x = EnzymeName, y= reorder(interaction(Genus, species, Strain), -Radius_um), size = ROSGene_count, color = Taxa, shape = Ome)) +
  scale_x_discrete(position ="top") +
  scale_color_manual(values = TaxaColors) +
  scale_shape_manual(values = c(16,1))+
  scale_radius(breaks=c(2,4,6,8,10,12,15,20,25)) +
  ggtitle("SupOx") +
  theme_bw() +
  # theme(axis.text.x = element_text(angle = 90, size = 10),
  #       axis.text.y = element_text(face="italic",size = 10, vjust = 0)) +
  facet_grid(rows = vars(Taxa), cols = vars(SupOx), scales = "free", space = "free") +
  ggpubr::rotate_x_text(hjust = 0) #+
  # theme(panel.grid.major = element_blank(),
  #       panel.grid.minor = element_blank())
EukSupOxDotPlot
# saveRDS(EukSupOxDotPlot, file = file.path(Figures, "EukSupOxDotPlot.Rds"))
```

```{r ProkSupOxDotPlot, fig.height = 10, fig.width = 10}
ProkSupOxDotPlot <- MergedData_na %>%
  filter(SupOx == "Scavenging" | SupOx == "Production", 
         Taxa == "Prokaryote",
         ROSEnzyme_TotalCount != 0, 
         GeneModels_count != 0) %>%
  ggplot() +
  geom_point(aes(x = EnzymeName, y= reorder(interaction(Genus, species, Strain), -Radius_um), size = ROSGene_count, color = Genus, shape = Ome)) +
  scale_x_discrete(position ="top") +
  # scale_color_manual(values = TaxaColors) +
  scale_shape_manual(values = c(16,1))+
  scale_radius(breaks=c(1,2,3,4,5)) +
  ggtitle("SupOx") +
  theme_bw() +
  # theme(axis.text.x = element_text(angle = 90, size = 10),
  #       axis.text.y = element_text(face="italic",size = 10, vjust = 0)) +
  facet_grid(rows = vars(Taxa), cols = vars(SupOx), scales = "free", space = "free") +
  ggpubr::rotate_x_text(hjust = 0) #+
  # theme(panel.grid.major = element_blank(),
  #       panel.grid.minor = element_blank())
# ProkSupOxDotPlot
# saveRDS(ProkSupOxDotPlot, file = file.path(Figures, "ProkSupOxDotPlot.Rds"))
```

```{r EukNitOxDotPlot, fig.height = 20, fig.width = 10}
EukNitOxDotPlot <- MergedData_na %>%
  filter(NitOx == "Scavenging" | NitOx == "Production", 
         Taxa != "Prokaryote",
         ROSEnzyme_TotalCount != 0, 
         GeneModels_count != 0) %>%
  ggplot() +
  geom_point(aes(x = EnzymeName, y= reorder(interaction(Genus, species, Strain), -Radius_um), size = ROSGene_count, color = Taxa, shape = Ome)) +
  scale_x_discrete(position ="top") +
  scale_color_manual(values = TaxaColors) +
  scale_shape_manual(values = c(16,1))+
  scale_radius(breaks=c(2,4,6,8,10,12,15,20,25)) +
  ggtitle("NitOx") +
  theme_bw() +
  # theme(axis.text.x = element_text(angle = 90, size = 10),
  #       axis.text.y = element_text(face="italic",size = 10, vjust = 0)) +
  facet_grid(rows = vars(Taxa), cols = vars(NitOx), scales = "free", space = "free") +
  ggpubr::rotate_x_text(hjust = 0) #+
  # theme(panel.grid.major = element_blank(),
  #       panel.grid.minor = element_blank())
EukNitOxDotPlot
# saveRDS(EukSupOxDotPlot, file = file.path(Figures, "EukSupOxDotPlot.Rds"))
```

```{r ProkNitOxDotPlot, fig.height = 10, fig.width = 10}
ProkNitOxDotPlot <- MergedData_na %>%
  filter(NitOx == "Scavenging" | NitOx == "Production", 
         Taxa == "Prokaryote",
         ROSEnzyme_TotalCount != 0, 
         GeneModels_count != 0) %>%
  ggplot() +
  geom_point(aes(x = EnzymeName, y= reorder(interaction(Genus, species, Strain), -Radius_um), size = ROSGene_count, color = Genus, shape = Ome)) +
  scale_x_discrete(position ="top") +
  # scale_color_manual(values = TaxaColors) +
  scale_shape_manual(values = c(16,1))+
  scale_radius(breaks=c(1,2,3,4,5)) +
  ggtitle("SupOx") +
  theme_bw() +
  # theme(axis.text.x = element_text(angle = 90, size = 10),
  #       axis.text.y = element_text(face="italic",size = 10, vjust = 0)) +
  facet_grid(rows = vars(Taxa), cols = vars(NitOx), scales = "free", space = "free") +
  ggpubr::rotate_x_text(hjust = 0) #+
  # theme(panel.grid.major = element_blank(),
  #       panel.grid.minor = element_blank())
# ProkNitOxDotPlot
# saveRDS(ProkSupOxDotPlot, file = file.path(Figures, "ProkSupOxDotPlot.Rds"))
```

XXX likely worth omitting

## Supplementary Tables {.unnumbered}

```{r DataDictionary, echo = FALSE}
options(knitr.kable.NA = '')
DataDictionary <- read_csv(file.path(DataIn, "DataDictionary.csv"))

DataDictionary %>% 
  select(Variable, Units, Definition, CodeLocation) %>%
  kable(caption = "Variable names, definitions, units, and first location of occurence in code, used for our data.") %>%
  kable_classic()
```

```{r ROSEnzymeList, echo = FALSE}
options(knitr.kable.NA = '')
ROSEnzymeList <- googlesheets4::read_sheet("https://docs.google.com/spreadsheets/d/1haHVI5bFVjBGPNW8YTCvCBecWKdrvOeocT5WfIYWnk0/edit#gid=903519796")  

ROSEnzymeList %>%
  select(ECNumber, KO, EnzymeName, SupOx, HyPe, NitOx) %>%
  kable(caption = "Enzyme Commission Number, Kegg Orthology Number, Enzyme Name and ROS Substrate Metabolised") %>%
  kable_classic()
```

```{r MetaData, echo = FALSE}
options(knitr.kable.NA = '')
MetaData <- googlesheets4::read_sheet("https://docs.google.com/spreadsheets/d/1kXm0GvMdTRbF12cG5Bps-2e_4MTjgUX7iUOjqiu-_c0/edit#gid=0") %>%
  # dplyr::select(c(Genus, species, Strain, FileName, Ome, Taxa, Rad1_um, Rad2_um, Rad3_um, Flagella, GenomeSize_mbp, GeneModels_count, Latitude, Longitude, Marine, PennateCentric, ColonySpecies)) %>% 
  mutate(Strain = as.character(Strain)) %>%
  select(-c(Rad_ref, LogLat_ref, ColonyCitation_other))

MetaData %>%
  kable(caption = glue("Metadata for each organism")) %>%
  kable_classic()
```

```{r ManualAutoCompareData, echo = FALSE}
options(knitr.kable.NA = '')
ManualAutoCompareData %>%
  select(-Automated_count) %>%
  rename(Automated_count = AutomatedECKO_count) %>%
  kable(caption = glue("Comparison of Manual and Automated Gene Counts")) %>%
  kable_classic()
```

# References {.unnumbered}
